Index: noatun-plugins/noatunmadness/madness.plugin
===================================================================
--- noatun-plugins/noatunmadness/madness.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/noatunmadness/madness.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -20,6 +20,7 @@
 Name[fi]=Hulluus
 Name[gl]=Loucura
 Name[hi]=मेडनेस
+Name[hr]=Ludilo
 Name[km]=ភាពឆ្កួត
 Name[lt]=Beprotybė
 Name[mk]=Беснило
@@ -59,9 +60,11 @@
 Comment[fa]=دامنۀ رنجش‌آور و مرموزی که پنجره‌های شما را حرکت می‌دهد
 Comment[fi]=Outo ja rasittava oskiloskooppi, joka liikuttelee ikkunoitasi
 Comment[fr]=Un aperçu magique et ennuyeux qui déplace votre fenêtre
+Comment[ga]=Scóp corr ciapach a bhogann do chuid fuinneog
 Comment[gl]=Un osciloscópio estraño e irritante que move as suas fiestras
 Comment[he]=תחום מוזר ומעצבן שמזיז את החלונות שלך
 Comment[hi]=एक सनकी और चिढ़ाकू स्कोप जो आपके विंडो को खिसकाता है
+Comment[hr]=Neobičan i živcirajući osciloskop koji pomiče vaše prozore
 Comment[hu]=Egy különleges ablakmozgató vizualizációs modul
 Comment[is]=Undarlegur og pirrandi mælir sem færir gluggana þína
 Comment[it]=Uno strano e scocciante "scope" che muove le tue finestre
Index: noatun-plugins/luckytag/luckytag.plugin
===================================================================
--- noatun-plugins/luckytag/luckytag.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/luckytag/luckytag.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -23,6 +23,7 @@
 Name[gl]=Leitor de Marcas Lucky
 Name[he]=קורא תגיות בר־מזל
 Name[hi]=लकी टैग रीडर
+Name[hr]=Lucky čitač oznaka
 Name[is]=Heppni merkingalesarinn
 Name[it]=Lettore Tag Lucky
 Name[ja]=Lucky タグリーダ
@@ -74,6 +75,7 @@
 Comment[gl]=Adiviña a información das marcas a partir do nome do ficheiro
 Comment[he]=ניחוש מידע התגיות מתוך שם הקובץ
 Comment[hi]=फ़ाइल-नामों से टैग जानकारी का अंदाजा लगाता है
+Comment[hr]=Nagađa podatke u oznaci iz naziva datoteke
 Comment[hu]=Tag-információ a fájlnév alapján
 Comment[is]=Giskar á lagamerkingar útfrá skráarheitum
 Comment[it]=Indovina le informazioni dei tag dal nome del file
Index: noatun-plugins/nexscope/nexscope.plugin
===================================================================
--- noatun-plugins/nexscope/nexscope.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/nexscope/nexscope.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -34,6 +34,7 @@
 Comment[gl]=A visualización ultra-configurábel
 Comment[he]=המחשה הניתנת להתאמה אישית מרבית
 Comment[hi]=एक अल्ट्रा-कस्टमाइजेबल विज़ुअलाइज़ेशन
+Comment[hr]=Totalno prilagodiva vizualizacija
 Comment[hu]=Százféle módon variálható vizualizációs modul
 Comment[is]=Sjónræm myndbrella sem er mjög stillanleg
 Comment[it]=Un visualizzatore ultra personalizzabile
Index: noatun-plugins/wavecapture/wavecapture.plugin
===================================================================
--- noatun-plugins/wavecapture/wavecapture.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/wavecapture/wavecapture.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -66,6 +66,7 @@
 Comment[fa]=گیراندازی پروندۀ wave
 Comment[fi]=Kaappaa äänitiedosto
 Comment[fr]=Capturer vers un fichier wave
+Comment[ga]=Gabháil i gcomhad toinne
 Comment[gl]=Capturar a un ficheiro WAVE
 Comment[he]=לכידה לקובץ wave
 Comment[hi]=वेव फ़ाइल में केप्चर करता है
Index: noatun-plugins/alarm/wakeup.plugin
===================================================================
--- noatun-plugins/alarm/wakeup.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/alarm/wakeup.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -67,6 +67,7 @@
 Comment[gl]=Unha alarma simples e configurábel
 Comment[he]=תזכורת פשוטה הניתנת להתאמה אישית
 Comment[hi]=एक साधारण, कस्टमाइजेबल अलार्म
+Comment[hr]=Jednostavan i prilagodljiv alarm
 Comment[hu]=Egyszerű, testreszabható emlékeztetés
 Comment[is]=Einfaldur og stillanlegur vekjari
 Comment[it]=Un allarme semplice e personalizzabile
Index: noatun-plugins/charlatan/charlatanui.plugin
===================================================================
--- noatun-plugins/charlatan/charlatanui.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/charlatan/charlatanui.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -34,8 +34,10 @@
 Comment[eu]=Hilda dagoen gizonaren interfazea
 Comment[fi]=Kuolleen miehen liittymä
 Comment[fr]=L'interface de l'homme mort
+Comment[ga]=Comhéadan na marbh
 Comment[gl]=A interface do home morto
 Comment[hi]=द डीड मैन्स इंटरफेस
+Comment[hr]=Sučelje mrtve straže
 Comment[hu]=Egy haláli vizualizációs modul
 Comment[is]=Viðmót dauða mannsins
 Comment[it]=L'interfaccia per l'uomo duro
Index: noatun-plugins/alsaplayer/alsaplayerui.plugin
===================================================================
--- noatun-plugins/alsaplayer/alsaplayerui.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/alsaplayer/alsaplayerui.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -40,6 +40,7 @@
 Comment[gl]=Clone da interface de AlsaPlayer
 Comment[he]=תואם ממשק המשתמש של AlsaPlayer
 Comment[hi]=अल्सा-प्लेयर यूआई क्लोन
+Comment[hr]=AlsaPlayer UI klon
 Comment[hu]=AlsaPlayer-változat
 Comment[is]=AlsaPlayer eftirlíking
 Comment[it]=Clone UI di AlsaPlayer
Index: noatun-plugins/blurscope/blurscope.plugin
===================================================================
--- noatun-plugins/blurscope/blurscope.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/blurscope/blurscope.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -36,6 +36,7 @@
 Comment[gl]=Un osciloscópio bonito e que se desvanece
 Comment[he]=מונוסקופ יפה שנמוג
 Comment[hi]=एक सुन्दर, मन्द होता मोनोस्कोप
+Comment[hr]=Lijep, pretapajući monoskop
 Comment[hu]=Elhalványuló monoszkóp
 Comment[is]=Falleg monoscope myndbrella
 Comment[ja]=感じの良いフェードするモノスコープ
Index: noatun-plugins/tyler/tyler.plugin
===================================================================
--- noatun-plugins/tyler/tyler.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/tyler/tyler.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -6,6 +6,7 @@
 License=GPL
 Name=Tyler
 Name[hi]=टायलर
+Name[hr]=Slagač
 Name[it]=Tessuto
 Name[lt]=Tyler 
 Name[ru]=Тайлер
@@ -31,6 +32,7 @@
 Comment[gl]=Portado a partir de Infinity de XMMS
 Comment[he]=הוסב מ־Infinity מ־XMMS
 Comment[hi]=इनफिनिटी से एक्सएमएमएस पर पोर्टेड
+Comment[hr]=Prebačeno iz Infinity-a, iz XMMS-a
 Comment[hu]=Az XMMS Infinity átirata
 Comment[is]=Flutt úr Infinity frá XMMS
 Comment[it]=Preso da Infinity di XMMS
Index: noatun-plugins/oblique/oblique.plugin
===================================================================
--- noatun-plugins/oblique/oblique.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/oblique/oblique.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -32,9 +32,11 @@
 Comment[fa]=فهرست پخش تلفیق خودکار
 Comment[fi]=Automaattisesti järjestyvä soittolista
 Comment[fr]=Liste de lecture auto-assemblée
+Comment[ga]=Seinmliosta uath-chomhordaithe
 Comment[gl]=Lista de reproduición auto-incorporante
 Comment[he]=רשימת ניגון נסגרת עצמאית
 Comment[hi]=ऑटो-कोलेटिंग प्लेलिस्ट
+Comment[hr]=Album s automatskim uspoređivanjem
 Comment[hu]=Automatikusan szelektáló lejátszási lista
 Comment[is]=Sjálfvirkur lagalisti
 Comment[it]=Raccoglie automaticamente la lista di selezione
Index: noatun-plugins/ffrs/ffrs.plugin
===================================================================
--- noatun-plugins/ffrs/ffrs.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/ffrs/ffrs.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -21,6 +21,7 @@
 Name[ga]=Réigiún Coigríochach na Fraince
 Name[gl]=Rexión Estranxeira da Franza
 Name[hi]=फ्रांसीसी विदेशी क्षेत्र
+Name[hr]=Francuska Legija stranaca
 Name[is]=Franska útlendingasveitin
 Name[it]=Legione Straniera Francese
 Name[km]=តំបន់​​ខាង​ក្រៅ​​ ភាសាបារាំង
@@ -67,6 +68,7 @@
 Comment[gl]=Osciloscópios flutuantes
 Comment[he]=תחומים צפים
 Comment[hi]=फ्लोटिंग स्कोप्स
+Comment[hr]=Plivajući oscilograf
 Comment[hu]=Lebegő monoszkópok
 Comment[is]=Fljótandi mælar
 Comment[it]=Floating Scopes
Index: noatun-plugins/lyrics/lyrics.plugin
===================================================================
--- noatun-plugins/lyrics/lyrics.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/lyrics/lyrics.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -22,6 +22,7 @@
 Name[gl]=Letras
 Name[he]=מילים
 Name[hi]=लिरिक्स
+Name[hr]=Tekstovi pjesama
 Name[it]=Testi
 Name[km]=អត្ថបទ​ចម្រៀង
 Name[lt]=Lyrika
@@ -66,9 +67,11 @@
 Comment[fa]=نمایش آهنگ برای ترانۀ جاری
 Comment[fi]=Näyttää kappaleen sanat
 Comment[fr]=Afficheur des paroles de la chanson en cours
+Comment[ga]=Taispeáin liricí an amhráin reatha
 Comment[gl]=Mostra as letras da peza actual
 Comment[he]=הצגת מילות השיר הנוכחי
 Comment[hi]=मौज़ूदा गीत का काव्य दिखाता है
+Comment[hr]=Prikazivanje teksta trenutne pjesme
 Comment[hu]=Megjeleníti az aktuális szám szövegét
 Comment[is]=Sýnir texta lagsins
 Comment[it]=Mostra il testo della canzione corrente
Index: noatun-plugins/synaescope/synaescope.plugin
===================================================================
--- noatun-plugins/synaescope/synaescope.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/synaescope/synaescope.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -8,6 +8,7 @@
 Name[da]=Synaeskop
 Name[de]=Synaeskop
 Name[hi]=सिनेस्कोप
+Name[hr]=Sinaeskop
 Name[nl]=Synaescoop
 Name[ru]=Зрительный образ Synaescope
 Name[sv]=Synaeskop
@@ -33,9 +34,11 @@
 Comment[fa]=تجسم Noatun
 Comment[fi]=Noatun visualisaatio
 Comment[fr]=Affichage de Noatun
+Comment[ga]=Amharcléiriú Noatun
 Comment[gl]=Visualización de Noatun
 Comment[he]=המחשה של Noatun
 Comment[hi]=नोआट्यून विज़ुअलाइज़ेशन
+Comment[hr]=Vizualizacija za Noatun
 Comment[hu]=Noatun vizuális modul
 Comment[is]=Sjónræn brella fyrir Noatun
 Comment[it]=Visualizzazione di Noatun
Index: noatun-plugins/tippercanoe/tippecanoe.plugin
===================================================================
--- noatun-plugins/tippercanoe/tippecanoe.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/tippercanoe/tippecanoe.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -30,9 +30,11 @@
 Comment[fa]=تجسمی که می‌توانید ساعتها اجرا کنید.
 Comment[fi]=Visuaalisaatio, jota voit soittaa tunteja
 Comment[fr]=Un affichage avec lequel vous pouvez jouer pendant des heures
+Comment[ga]=Amharcléiriú ar féidir súgradh leis ar feadh uaireanta an chloig.
 Comment[gl]=Unha visualización que o usuário pode reproducir durante horas.
 Comment[he]=המחשה שאפשר לשחק איתה שעות
 Comment[hi]=एक विज़ुअलाइज़ेशन जिसके साथ आप घंटों खेल सकते हैं
+Comment[hr]=Vizualizacija koju možete gledati satima
 Comment[hu]=Egy sokoldalú vizualizációs modul
 Comment[is]=Sjónræn brella sem hægt er að leika sér með lengi.
 Comment[it]=Una visualizzazione con cui puoi giocare per ore
Index: noatun-plugins/dub/dub/dub.plugin
===================================================================
--- noatun-plugins/dub/dub/dub.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/dub/dub/dub.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -24,9 +24,11 @@
 Comment[fa]=وصلۀ فهرست پخش Dub
 Comment[fi]=Dub soittolistasovelma
 Comment[fr]=Module de liste de doublage
+Comment[ga]=Breiseán seinmliosta Dub
 Comment[gl]=Plugin de lista de reproduición de Dub
 Comment[he]=תוסף רשימת הניגון Dub
 Comment[hi]=डब प्लेलिस्ट प्लगइन
+Comment[hr]=Dodatak albuma za Dub
 Comment[hu]=Dub lejátszási lista modulként
 Comment[is]=Tól til að spila Dub lagalista
 Comment[it]=Plugin per le Playlist di Dub
Index: noatun-plugins/pitchablespeed/pitchablespeed.plugin
===================================================================
--- noatun-plugins/pitchablespeed/pitchablespeed.plugin	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ noatun-plugins/pitchablespeed/pitchablespeed.plugin	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -22,6 +22,7 @@
 Name[gl]=Velocidade Variábel de Reprodución
 Name[he]=התאמת מהירות
 Name[hi]=पिचेबल प्लेइंग स्पीड
+Name[hr]=Promjenjiva brzina
 Name[hu]=Változtatható lejátszási sebesség
 Name[is]=Stillanlegur afspilunarhraði
 Name[it]=Modificatore di velocità di riproduzione
@@ -63,12 +64,13 @@
 Comment[es]=Permite ajustar la velocidad de reproducción con un botón deslizante
 Comment[et]=Muudab mängimiskiiruse liuguriga kohandatavaks
 Comment[eu]=Erreprodukzio abiadura botoi irristagarri baten bitartez doitzen du
-Comment[fa]=سرعت پخش را توسط یک لغزنده، قابل تنظیم می‌سازد
+Comment[fa]=سرعت پخش را توسط یک لغزان قابل تنظیم می‌سازد
 Comment[fi]=Voit muuttaa soittonopeutta liukusäätimellä
 Comment[fr]=Rend la vitesse de lecture ajustable avec un curseur
 Comment[gl]=Torna a velocidade de reproduición axustábel cunha barra
 Comment[he]=מאפשר התאמה של מהירות הניגון בעזרת מחוון
 Comment[hi]=प्लेइंग स्पीड एक स्लाइडर के द्वारा समायोजन योग्य बनाता है
+Comment[hr]=Omogućuje promjenu brzine sviranja pomoću klizača
 Comment[hu]=Változtatható lejátszási sebességet tesz lehetővé
 Comment[is]=Gerir afspilunarhraðan stillanlegan með rennislá
 Comment[it]=Permette di modificare la velocità di riproduzione tramite una barra
Index: kate/xmltools/katexmltools.desktop
===================================================================
--- kate/xmltools/katexmltools.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/xmltools/katexmltools.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -78,6 +78,7 @@
 Comment[fi]=Listaa XML-elementit, attribuutit, attribuuttiarvot ja DTD:n sallimat entityt
 Comment[fo]=Vísir XML-eindir, eginleikir, eginleikisvirði og styttingar loyvdir av skjalaslagnum (DTD'in).
 Comment[fr]=Liste les éléments, attributs, valeurs d'attribut et entités XML autorisés par une DTD
+Comment[ga]=Liostaigh eilimintí XML, aitreabúidí, luachanna aitreabúide agus aonáin atá ceadaithe ag an DTD
 Comment[gl]=Lista os elementos, atributos, valores de atributos e entidades permitidas en XML polo DTD
 Comment[he]=הצגת רשימה של מרכיבים, מאפיינים, ערכי מאפיינים וישויות XML שה־DTD מתיר
 Comment[hi]=डीटीडी के द्वारा स्वीकार्य एक्सएमएल अवयवों, एट्रिब्यूट्स, एट्रिब्यूट मूल्य तथा एंटिटीस को सूचीबद्ध करता है
Index: kate/scripts/html-tidy.desktop
===================================================================
--- kate/scripts/html-tidy.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/scripts/html-tidy.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -4,7 +4,7 @@
 Name[cs]=Úklid HTML
 Name[cy]=Tacluso HTML (HTML Tidy)
 Name[es]=Ordenado de HTML
-Name[fa]= Tidy زنگام
+Name[fa]=Tidy زنگام
 Name[hi]=एचटीएमएल टाइडी
 Name[hr]=HTML preuređivanje
 Name[lt]=HTML valymas
@@ -31,9 +31,11 @@
 Comment[fa]=سند جاری را از طریق Tidy زنگام اعتبارسنجی می‌کند
 Comment[fi]=Validoi nykyisen asiakirjan HTML Tidyllä
 Comment[fr]=Valide le document courant grâce à HTML Tidy
+Comment[ga]=Bailíochtaigh an cháipéis reatha le HTML Tidy
 Comment[gl]=Valida o documento actual utilizando HTML Tidy
 Comment[he]=מוודא את המסמך הנוכחי באמצעות HTML Tidy
 Comment[hi]=एचटीएमएल टाइडी के द्वारा मौज़ूदा दस्तावेज़ वेलिडेट करता है
+Comment[hr]=Provjera valjanosti trenutnog dokumenta putem aplikacije HTML Tidy
 Comment[hu]=Az aktuális dokumentum ellenőrzése a HTML Tidy programmal
 Comment[is]=Staðfestir núverandi skjal með HTML Tidy
 Comment[it]=Convalida il documento attuale tramite HTML Tidy
Index: kate/tabbarextension/katetabbarextension.desktop
===================================================================
--- kate/tabbarextension/katetabbarextension.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/tabbarextension/katetabbarextension.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -16,6 +16,7 @@
 Name[fa]=توسعۀ میله تب Kate
 Name[fi]=Kate tabulointipalkkilaajennus
 Name[fr]=Extension de la barre d'onglets pour Kate
+Name[ga]=Eisínteacht Barra Cluaisíní Kate
 Name[gl]=Extensión de Barra de Páxina para Kate
 Name[hi]=के-एटीई टैब पट्टी एक्सटेंशन
 Name[hr]=Kate proširenje za trake kartica
@@ -62,8 +63,10 @@
 Comment[fa]=میله تب را به پنجرۀ اصلی Kate اضافه می‌کند.
 Comment[fi]=Lisää tabulointipalkin Katen pääikkunaan
 Comment[fr]=Ajoute une barre d'onglet dans la fenêtre principale de Kate
+Comment[ga]=Cuir barra cluaisíní le príomhfhuinneog Kate
 Comment[gl]=Engade unha barra de páxinas á fiestra principal de Kate
 Comment[hi]=केएटीई के मुख्य विंडो में एक टैब पट्टी जोड़ता है
+Comment[hr]=Dodaje traku kartica u glavni prozor aplikacije Kate
 Comment[hu]=Hozzáad egy lapozósávot a Kate főablakához
 Comment[is]=Bætir við flipaslá í aðalglugga Kate
 Comment[it]=Aggiunge una barra delle linguette alla finestra principale di Kate
Index: kate/kjswrapper/samples/katekjsconsolewindow.desktop
===================================================================
--- kate/kjswrapper/samples/katekjsconsolewindow.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/kjswrapper/samples/katekjsconsolewindow.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -17,6 +17,7 @@
 Name[fa]=پنجرۀ پیشانۀ جاوااسکریپت Kate
 Name[fi]=Kate Javascript konsoli-ikkuna
 Name[fr]=Console Javascript pour Kate
+Name[ga]=Fuinneog Chonsóil Javascript Kate
 Name[gl]=Fiestra de Consola de JavaScript para Kate 
 Name[he]= חלון תסריטי הג'אווה של Kate
 Name[hi]=केएटीई जावास्क्रिप्ट कंसोल विंडो
Index: kate/kjswrapper/samples/katekjsselect.desktop
===================================================================
--- kate/kjswrapper/samples/katekjsselect.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/kjswrapper/samples/katekjsselect.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -18,6 +18,7 @@
 Name[fr]=Sélection du plus petit bloc enfermé
 Name[gl]=Seleccionar o Menor Bloco Envolvente
 Name[he]=בחר את בלוק הסגירה הכי קטן
+Name[hr]=Odabir najmanjeg zatvorenog bloka
 Name[hu]=A legkisebb bezáró blokk kijelölése
 Name[is]=Velja minnstu umluktu blokkina
 Name[it]=Seleziona il più piccolo blocco
Index: kate/kjswrapper/samples/katekjstest1.desktop
===================================================================
--- kate/kjswrapper/samples/katekjstest1.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/kjswrapper/samples/katekjstest1.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -47,9 +47,10 @@
 Comment[es]=Test para el ajuste KJS
 Comment[et]=KJS Wrapperi test
 Comment[eu]=KJS Wrapper-erako testua
-Comment[fa]=آزمون برای سطرشکن KJS
+Comment[fa]=آزمون برای سطربندی KJS
 Comment[fi]=KJS Wrapperin testi
 Comment[fr]=Test pour l'interface avec KJS
+Comment[ga]=Tástáil an Rapar KJS
 Comment[gl]=Proba para o Wrapper de KJS
 Comment[he]=ניסיון עבור המעטפת KJS
 Comment[hi]=केजेएस व्रेपर के लिए जाँच
Index: kate/cppsymbolviewer/katecppsymbolviewer.desktop
===================================================================
--- kate/cppsymbolviewer/katecppsymbolviewer.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/cppsymbolviewer/katecppsymbolviewer.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -18,6 +18,7 @@
 Name[fa]=مشاهده‌گر نماد Kate C/C++
 Name[fi]=Katen C/C++-symboleiden näyttäjä
 Name[fr]=Visualiseur de symboles C/C++ pour Kate
+Name[ga]=Amharcán Siombailí C/C++ le haghaidh Kate
 Name[gl]=Visor de Símbolos C/C++ para Kate
 Name[he]=מציג סמלים של C/C++ עבור Kate
 Name[hi]=के-एटीई C/C++ प्रतीक प्रदर्शक
@@ -68,9 +69,11 @@
 Comment[fa]=این وصله نمادهای C/C++ را از متن استخراج کرده و نمایش می‌دهد
 Comment[fi]=Tämä liitännäinen hakee ja näyttää C/C++-symbolit lähdekoodista
 Comment[fr]=Ce module extrait et affiche les symboles C/C++ du source
+Comment[ga]=Baineann an breiseán seo siombailí C/C++ as cód foinseach agus taispeánann sé iad
 Comment[gl]=Este plugin extrai e mostra símbolos C/C++ do código fonte
 Comment[he]= תוסף זה מוציא ומראה סמלים מתוך קבצי C/C++ 
 Comment[hi]=यह प्लगइन स्रोत में से C/C++ प्रतीकों को उद्धृत कर प्रदर्शित करता है 
+Comment[hr]=Dodatak za izvlačenje i prikazivanje simbola C/C++ iz izvornog koda
 Comment[hu]=Ezzel a bővítőmodullal C/C++-os forrásfájlokból lehet kilistázni a szimbólumokat
 Comment[is]=Þetta íforrit sækir og birtir C/C++ tákn úr frumkóða
 Comment[it]=Questo plugin estrae e mostra simboli C/C++ dal sorgente
Index: kate/modeline/katemodeline.desktop
===================================================================
--- kate/modeline/katemodeline.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/modeline/katemodeline.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -36,7 +36,7 @@
 Name[nds]=Modeline-Moduul för Kate
 Name[nl]=Kate Modeline-plugin
 Name[nn]=Kate nivålinje-tillegg
-Name[pa]=ਕੇਟ ਢੰਗ-ਸਤਰ ਪਲੱਗਿੰਨ
+Name[pa]=ਕੇਟ ਢੰਗ-ਸਤਰ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka ustawień linii do Kate
 Name[pt]='Plugin' de Linha de Modo do Kate
 Name[pt_BR]=Plugin Modeline do Kate
Index: kate/filetemplates/plugin/katefiletemplates.desktop
===================================================================
--- kate/filetemplates/plugin/katefiletemplates.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/filetemplates/plugin/katefiletemplates.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -65,6 +65,7 @@
 Comment[fa]=ایجاد پرونده‌های جدید از قالبهای درون ویرایشگر متن
 Comment[fi]=Luo uudet tiedostot Kate-tekstieditorin malleista
 Comment[fr]=Création de nouveaux fichiers à partir de modèles dans l'éditeur de texte de Kate
+Comment[ga]=Cruthaigh comhaid nua ó theimpléid in eagarthóir téacs Kate
 Comment[gl]=Criar novos ficheiros a partir de modelos no editor de textKate
 Comment[hi]=टैम्पलेट्स के द्वारा केएटीई पाठ संपादक में नया फ़ाइल बनाएँ
 Comment[hr]=Izrađuje nove datoteke iz predložaka unutar Kate uređivača
Index: kate/xmlcheck/katexmlcheck.desktop
===================================================================
--- kate/xmlcheck/katexmlcheck.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/xmlcheck/katexmlcheck.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -72,6 +72,7 @@
 Comment[fa]=پروندۀ XML را با استفاده از xmllint اعتبارسنجی می‌کند
 Comment[fi]=Validoi XML tiedostoja käyttäen xmllint ohjelmaa
 Comment[fr]=Valide un fichier XML en utilisant « xmllint »
+Comment[ga]=Bailíochtaigh comhaid XML le xmllint
 Comment[gl]=Valida ficheiros XML utilizando xmllint
 Comment[he]=וידוא קבצי XML באמצעות xmllint
 Comment[hi]=xmllint के उपयोग से XML फ़ाइलों को वेलिडेट करता है
Index: kate/filelistloader/katefll_plugin.desktop
===================================================================
--- kate/filelistloader/katefll_plugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/filelistloader/katefll_plugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -19,6 +19,7 @@
 Name[fa]=بارکنندۀ فهرست پروندۀ Kate
 Name[fi]=Kate tiedostolistalatain
 Name[fr]=Chargeur de liste de fichiers pour Kate
+Name[ga]=Luchtóir Liostaí Comhad Kate
 Name[gl]=Leitor da Lista de Ficheiros para Kate
 Name[he]=טוען רשימות הקבצים של Kate
 Name[hi]=के-एटीई फ़ाइल सूची लोड करने वाला
@@ -73,6 +74,7 @@
 Comment[fa]=فهرست پروندۀ Kate را بر روی دیسک ذخیره می‌کند.
 Comment[fi]=Tallenna Katen tiedostolista levylle
 Comment[fr]=Enregistrement de la liste des fichiers de Kate sur le disque
+Comment[ga]=Stóráil an liosta comhad Kate ar an diosca
 Comment[gl]=Guarda a lista de ficheiros para Kate no disco
 Comment[he]=שמירת רשימת הקבצים של Kate על גבי הכונן
 Comment[hi]=डिस्क पर के-एटीई की फ़ाइल सूची भंडारित करता है
Index: kate/filelistloader/katefll_initplugin.desktop
===================================================================
--- kate/filelistloader/katefll_initplugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/filelistloader/katefll_initplugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -15,6 +15,7 @@
 Name[fa]=وصلۀ آغازین Kate FLL
 Name[fi]=Kate FLL alustussovelma
 Name[fr]=Module externe FLL d'initialisation pour Kate
+Name[ga]=Breiseán Túsaithe FLL le haghaidh Kate
 Name[gl]=Plugin de Lista de Ficheiros para Kate
 Name[he]=תוסף אתחול FLL ל־Kate
 Name[hi]=के-एटीई एफएलएल इनिट प्लगइन
@@ -68,6 +69,7 @@
 Comment[gl]=Inicializa a lista de ficheiros abertos en Kate cos URLs guardados nun script inicial
 Comment[he]=אתחול רשימת הקבצים הפתוחים של Kate עם כתובות השמורות ב"תסריט אתחול"
 Comment[hi]=के-एटीई के फ़ाइल खोलें सूची इनिशियलाइज़ करता है जो कि उन यूआरएल के साथ एक "इनिट स्क्रिप्ट" में होते हैं
+Comment[hr]=Inicijalizira popis otvorenih datoteka aplikacije Kate s URL adresama pohranjenim unutar skripte "init script"
 Comment[hu]=Feltölti a Kate-ben a megnyitott fájlok listáját azokkal az URL-ekkel, melyek a megadott "inicializációs szkriptben" találhatók
 Comment[is]=Frumstillir opnar skrár Kate með lista af þeim slóðum sem eru geymdar í "init script"
 Comment[it]=Inizializza la lista dei file aperti da Kate con gli URL salvati in uno script di inizializzazione
Index: kate/insertcommand/kateinsertcommand.desktop
===================================================================
--- kate/insertcommand/kateinsertcommand.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/insertcommand/kateinsertcommand.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -69,6 +69,7 @@
 Comment[fi]=Lisää komentotulkin tuloste asiakirjaan
 Comment[fo]=Innset úttak frá eitt stýriboð/forrit í eitt skjal.
 Comment[fr]=Insertion de la sortie d'une commande shell dans un document
+Comment[ga]=Ionsáigh aschur ó ordú blaoisce isteach i gcáipéis
 Comment[gl]=Inserir o resultado da execución dun comando nun documento
 Comment[he]=הוספת פלט של פקודות מעטפת לתוך מסמך
 Comment[hi]=एक दस्तावेज़ में शेल कमांड आउटपुट प्रविष्ट करता है
Index: kate/snippets/katesnippets.desktop
===================================================================
--- kate/snippets/katesnippets.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/snippets/katesnippets.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -12,7 +12,7 @@
 Name[el]=Δείγματα Kate
 Name[es]=Recorte de Kate
 Name[et]=Kate koodijupid
-Name[fa]= تکه‌های Kate
+Name[fa]=تکه‌های Kate
 Name[fi]=Kate tekstipalat
 Name[fr]=Fragments de codes pour Kate
 Name[ga]=Blúirí Kate
Index: kate/kpybrowser/katepybrowse.desktop
===================================================================
--- kate/kpybrowser/katepybrowse.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/kpybrowser/katepybrowse.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -36,7 +36,7 @@
 Name[nb]=Pythonleser-programtillegg til Kate
 Name[nds]=Python-Kiekermoduul för Kate
 Name[nn]=Kate-programtillegg for Python-oversikt
-Name[pa]=ਕੇਟ ਪਾਈਥਨ ਝਲਕਾਰਾ ਪਲੱਗਿੰਨ
+Name[pa]=ਕੇਟ ਪਾਈਥਨ ਝਲਕਾਰਾ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka przeglądarki do Pythona do Kate
 Name[pt]='Plugin' de Navegação no Python do Kate
 Name[pt_BR]=Plug-in de Navegação Python do Kate
@@ -76,6 +76,7 @@
 Comment[fi]=Python selainsovelma Katelle
 Comment[fo]=Ein Python-kagari-ístingur til Kate.
 Comment[fr]=Un module externe de navigateur en Python pour Kate
+Comment[ga]=Breiseán brabhsála Python le haghaidh Kate
 Comment[gl]=Un plugin de navegación por Python para Kate
 Comment[he]=תוסף דפדפן Python ל־Kate
 Comment[hi]=के-एटीई के लिए पायथन ब्राउज़र प्लगइन
@@ -92,7 +93,7 @@
 Comment[nds]=En Python-Kiekermoduul för Kate
 Comment[nl]=Een Python-browserplugin voor Kate
 Comment[nn]=Ei Python-oversikt til Kate
-Comment[pa]=ਕੇਟ ਲਈ ਪਾਈਥਨ ਝਲਕਾਰਾ ਪਲੱਗਿੰਨ
+Comment[pa]=ਕੇਟ ਲਈ ਪਾਈਥਨ ਝਲਕਾਰਾ ਪਲੱਗਇਨ
 Comment[pl]=Wtyczka przeglądarki Pythona do Kate
 Comment[pt]=Um 'plugin' de navegação no Python para o Kate
 Comment[pt_BR]=Um Plugin de navegação Python para o Kate
Index: kate/openheader/kateopenheader.desktop
===================================================================
--- kate/openheader/kateopenheader.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/openheader/kateopenheader.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -61,6 +61,7 @@
 Comment[fi]=Avaa vastaavan .h/[.cpp|.c] tiedoston
 Comment[fo]=Opnar tann samsvarandi .h/[.cpp|.c]-fíla.
 Comment[fr]=Ouvre le fichier correspondant .h/[.cpp|.c]
+Comment[ga]=Oscail an comhad .h/[.cpp|.c] a fhreagraíonn leis an gceann seo
 Comment[gl]=Abre o ficheiro .h/[.cpp|.c] correspondente
 Comment[he]=פתיחת קובץ הכותרת המתאים [h/[.cpp|.c.
 Comment[hi]= .h/[.cpp|.c] के सदृश्य फ़ाइल खोलता है
Index: kate/helloworld/katehelloworld.desktop
===================================================================
--- kate/helloworld/katehelloworld.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/helloworld/katehelloworld.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -33,7 +33,7 @@
 Name[nds]=Mien eerst Kate-Moduul
 Name[nl]=Mijn eerste Kate-plugin
 Name[nn]=Mitt første programtillegg til Kate
-Name[pa]=ਮੇਰੀ ਪਹਿਲੀ ਕੇਟ ਪਲੱਗਿੰਨ
+Name[pa]=ਮੇਰੀ ਪਹਿਲੀ ਕੇਟ ਪਲੱਗਇਨ
 Name[pl]=Moja pierwsza wtyczka do Kate
 Name[pt]=O Meu Primeiro 'Plugin' para o Kate
 Name[pt_BR]=Meu Primeiro Plugin para o Kate
@@ -68,6 +68,7 @@
 Comment[fi]=Tässä pitäisi olla pieni kuvaus sovelmasta
 Comment[fo]=Her kann tú skriva ein lýsing av ístingurin.
 Comment[fr]=La description brève de votre module se place ici
+Comment[ga]=Cuir do chur síos gearr ar an mbreiseán anseo
 Comment[gl]=Aqui deberá colocar unha descrición do plugin
 Comment[he]=כאן אמור להיות התיאור הקצר שלך אודות התוסף
 Comment[hi]=प्लगइन के बारे में आपका संक्षिप्त वर्णन यहाँ जाएगा
Index: kate/make/katemake.desktop
===================================================================
--- kate/make/katemake.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kate/make/katemake.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -36,7 +36,7 @@
 Name[nds]=Make-Moduul för Kate
 Name[nl]=Kate Make-plugin
 Name[nn]=Kate Make-programtillegg
-Name[pa]=ਕੇਟ Make ਪਲੱਗਿੰਨ
+Name[pa]=ਕੇਟ Make ਪਲੱਗਇਨ
 Name[pl]=Wtyczka Make dla Kate
 Name[pt]='Plugin' de Make do Kate
 Name[pt_BR]=Plugin de Make do Kate
Index: konq-plugins/mediarealfolder/media_realfolder.desktop
===================================================================
--- konq-plugins/mediarealfolder/media_realfolder.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/mediarealfolder/media_realfolder.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -19,6 +19,7 @@
 Name[fr]=Dossier système d'ouverture de média
 Name[gl]=Cartafol de Sistema de Médio Aberto
 Name[he]=פתח את תיקיית מדיית המערכת
+Name[hr]=Otvaranje srednjih mapa sustava
 Name[hu]=A médiaanyagok rendszerkönyvtárának megnyitása
 Name[is]=Opin miðils kerfismappa
 Name[it]=Cartella di sistema per l'apertura di supporti
Index: konq-plugins/babelfish/plugin_babelfish.desktop
===================================================================
--- konq-plugins/babelfish/plugin_babelfish.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/babelfish/plugin_babelfish.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -41,7 +41,7 @@
 Name[nds]=Babel-Moduul
 Name[nl]=Babel-plugin
 Name[nn]=Programtillegg for Babel
-Name[pa]=ਬਬੀਲ ਪਲੱਗਿੰਨ
+Name[pa]=ਬਬੀਲ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka programu Babel
 Name[pt]='Plugin' Babel
 Name[pt_BR]=Plug-in Babel
@@ -93,7 +93,7 @@
 Comment[nds]=Babelfish-Moduul
 Comment[nl]=Babelfish-plugin
 Comment[nn]=Programtillegg for Babelfish
-Comment[pa]=ਬਬੀਲਫਿਸ਼ ਪਲੱਗਿੰਨ
+Comment[pa]=ਬਬੀਲਫਿਸ਼ ਪਲੱਗਇਨ
 Comment[pl]=Wtyczka do programu Babelfish
 Comment[pt]='Plugin' do Babelfish
 Comment[pt_BR]=Plugin do Babelfish
Index: konq-plugins/akregator/akregator_konqfeedicon.desktop
===================================================================
--- konq-plugins/akregator/akregator_konqfeedicon.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/akregator/akregator_konqfeedicon.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -28,7 +28,7 @@
 Name[nds]=Konqueror-Moduul för't RSS-Stroomlüttbild
 Name[nl]=Konq Feed-pictogram-plugin
 Name[nn]=Programtillegg for kjeldeikon i Konqueror
-Name[pa]=Konq ਫੀਡ ਆਈਕਾਨ ਪਲੱਗਿੰਨ
+Name[pa]=Konq ਫੀਡ ਆਈਕਾਨ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka Konq z ikoną RSS
 Name[pt]='Plugin' de Ícone de Fonte do Konqueror
 Name[pt_BR]=Plug-in do Ícone de Fontes do Konq
@@ -59,7 +59,9 @@
 Comment[fa]=هنگامی که صفحه خوراندن RSS را دارد، شمایل خوراندن را در میله وضعیت نمایش می‌دهد
 Comment[fi]=Näyttää syötekuvakkeen tilapalkissa jos sivulla on RSS-syöte
 Comment[fr]=Affiche une icône de flux RSS dans la barre d'état lorsque la page en possède
+Comment[ga]=Taispeáin deilbhín fotha sa bharra stádais nuair atá fotha RSS ar an leathanach
 Comment[gl]=Mostra o ícone de fonte na barra de estado cando a páxina ten unha fonte RSS
+Comment[hr]=Prikazivanje ikone tekućih novosti u traci stanja ako web-stranica ima RSS novosti
 Comment[hu]=Csatornaikont jelenít meg az állapotsorban, ha az oldalhoz RSS-csatorna tartozik
 Comment[is]=Sýna straumtáknmynd í stöðuslá þegar síða inniheldur RSS straum
 Comment[it]=Mostra l'icona nella barra di stato quando la pagina ha una fonte RSS
Index: konq-plugins/domtreeviewer/plugin_domtreeviewer.desktop
===================================================================
--- konq-plugins/domtreeviewer/plugin_domtreeviewer.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/domtreeviewer/plugin_domtreeviewer.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -25,6 +25,7 @@
 Name[fa]=مشاهده‌گر درخت DOM
 Name[fi]=DOM-puun näyttäjä
 Name[fr]=Visualiseur d'arborescence DOM
+Name[ga]=Amharcán Crainn DOM
 Name[gl]=Visor da Árbore DOM
 Name[he]=מציג עץ DOM
 Name[hi]=डॉम ट्री प्रदर्शक
@@ -77,6 +78,7 @@
 Comment[fa]=مشاهده‌گر درخت DOM
 Comment[fi]=DOM-puun näyttäjä
 Comment[fr]=Afficheur d'arborescence de modèle orienté document
+Comment[ga]=Amharcán crainn DOM
 Comment[gl]=Visor da árbore DOM
 Comment[he]=מציג עץ DOM
 Comment[hi]=डॉम ट्री प्रदर्शक
Index: konq-plugins/microformat/mf_konqmficon.desktop
===================================================================
--- konq-plugins/microformat/mf_konqmficon.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/microformat/mf_konqmficon.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -12,6 +12,7 @@
 Name[fa]=وصلۀ ریزقالب Konqueror
 Name[fi]=Konqueror mikromuotoliitännäinen
 Name[fr]=Module d'icône de flux pour Konqueror
+Name[ga]=Breiseán Micreafhormáid Konqueror
 Name[gl]=Plugin de Micro-formato de Konqueror
 Name[he]=תוסף Microformat עבור Konqueror
 Name[hr]=Konqueror Microformat dodatak
@@ -26,7 +27,7 @@
 Name[nds]=Mikroformaat-Moduul för Konqueror
 Name[nl]=Konqueror Microformat-plugin
 Name[nn]=Programtillegg for mikroformat i Konqueror
-Name[pa]=ਕੋਨਕਿਊਰੋਰ ਮਾਈਕਰੋਫਾਰਮਿਟ ਪਲੱਗਿੰਨ
+Name[pa]=ਕੋਨਕਿਊਰੋਰ ਮਾਈਕਰੋਫਾਰਮਿਟ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka Mikroformatu dla Konquerora
 Name[pt]='Plugin' de Micro-formato do Konqueror
 Name[pt_BR]=Plug-in de fonte de notícias RSS do Konqueror
@@ -55,7 +56,9 @@
 Comment[fa]=وقتی صفحه دارای ریزقالب باشد، شمایل را در میله وضعیت نمایش می‌دهد
 Comment[fi]=Näyttää syötekuvakkeen tilapalkissa jos sivulla on mikromuoto
 Comment[fr]=Affiche une icône dans la barre d'état lorsque la page possède un microformat
+Comment[ga]=Taispeánann sé deilbhín sa bharra stádais nuair atá micreafhormáid ag an leathanach
 Comment[gl]=Mostra un ícone na barra de estado cando a páxina ten un micro-formato
+Comment[hr]=Prikazuje ikonu u traci stanja ako stranica posjeduje mikro-oblikovanje
 Comment[hu]=Ikont jelenít meg az állapotsorban, ha az oldalhoz található Microformat-információ
 Comment[is]=Sýnir táknmynd í stöðuslá þegar síða inniheldur microformat
 Comment[it]=Mostra l'icona nella barra di stato quando la pagina ha un microformat
Index: konq-plugins/rellinks/plugin_rellinks.desktop
===================================================================
--- konq-plugins/rellinks/plugin_rellinks.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/rellinks/plugin_rellinks.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -41,7 +41,7 @@
 Name[nds]=Dokmentbetöög-Moduul
 Name[nl]=Documentrelatiesplugin
 Name[nn]=Programtillegg for dokumentrelasjonar
-Name[pa]=ਦਸਤਾਵੇਜ਼ ਸਬੰਧ ਪਲੱਗਿੰਨ
+Name[pa]=ਦਸਤਾਵੇਜ਼ ਸਬੰਧ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka zależności w dokumencie
 Name[pt]='Plugin' de Relações de Documento
 Name[pt_BR]=Plug-in de Relações em Documentos
@@ -88,7 +88,7 @@
 Comment[nds]=Dokmentbetöög-Moduul
 Comment[nl]=Documentrelatiesplugin
 Comment[nn]=Programtillegg for dokumentrelasjonar
-Comment[pa]=ਦਸਤਾਵੇਜ਼ ਸਬੰਧ ਪਲੱਗਿੰਨ
+Comment[pa]=ਦਸਤਾਵੇਜ਼ ਸਬੰਧ ਪਲੱਗਇਨ
 Comment[pl]=Wtyczka zależności w dokumencie
 Comment[pt]='Plugin' de relações de documento
 Comment[pt_BR]=Plug-in para Relacionamentos entre Documentos
Index: konq-plugins/dirfilter/dirfilterplugin.desktop
===================================================================
--- konq-plugins/dirfilter/dirfilterplugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/dirfilter/dirfilterplugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -41,7 +41,7 @@
 Name[nds]=Ornerfilter-Moduul
 Name[nl]=Mappenfilterplugin
 Name[nn]=Progamtillegg for katalogfilter
-Name[pa]=ਡਾਇਰੈਕਟਰੀ ਫਿਲਟਰ ਪਲੱਗਿੰਨ
+Name[pa]=ਡਾਇਰੈਕਟਰੀ ਫਿਲਟਰ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka filtrowania katalogów
 Name[pt]='Plugin' de Filtragem de Directorias
 Name[pt_BR]=Pasta do Plug-in de Filtro
@@ -91,7 +91,7 @@
 Comment[nds]=Ornerfilter-Moduul
 Comment[nl]=Mappenfilterplugin
 Comment[nn]=Progamtillegg for katalogfilter
-Comment[pa]=ਡਾਇਰੈਕਟਰੀ ਫਿਲਟਰ ਪਲੱਗਿੰਨ
+Comment[pa]=ਡਾਇਰੈਕਟਰੀ ਫਿਲਟਰ ਪਲੱਗਇਨ
 Comment[pl]=Wtyczka filtrowania katalogów
 Comment[pt]='Plugin' de filtragem de directorias
 Comment[pt_BR]=Pasta de plug-in de filtragem
Index: konq-plugins/autorefresh/autorefresh.desktop
===================================================================
--- konq-plugins/autorefresh/autorefresh.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/autorefresh/autorefresh.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -39,7 +39,7 @@
 Name[nds]=Moduul för't automaatsche Opfrischen
 Name[nl]=Autoverversplugin
 Name[nn]=Programtillegg for automatisk oppfrisking
-Name[pa]=ਸਵੈਚਾਲਿਤ ਤਾਜ਼ਾ ਪਲੱਗਿੰਨ
+Name[pa]=ਸਵੈਚਾਲਿਤ ਤਾਜ਼ਾ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka automatycznego odświeżania
 Name[pt]='Plugin' de Actualização Automática
 Name[pt_BR]=Plug-in de atualização automática
@@ -86,7 +86,7 @@
 Comment[nds]=Moduul för't automaatsche Opfrischen
 Comment[nl]=Autoverversplugin
 Comment[nn]=Programtillegg for automatisk oppfrisking
-Comment[pa]=ਸਵੈ-ਚਾਲਿਤ ਤਾਜ਼ਾ ਪਲੱਗਿੰਨ
+Comment[pa]=ਸਵੈ-ਚਾਲਿਤ ਤਾਜ਼ਾ ਪਲੱਗਇਨ
 Comment[pl]=Wtyczka automatycznego odświeżania
 Comment[pt]='Plugin' de actualização automática
 Comment[pt_BR]=Plug-in de atualização automática
Index: konq-plugins/searchbar/searchbar.desktop
===================================================================
--- konq-plugins/searchbar/searchbar.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/searchbar/searchbar.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -26,6 +26,7 @@
 Name[fa]=وصلۀ میلۀ جستجو
 Name[fi]=Etsintäpalkki-liitännäinen
 Name[fr]=Module de barre de recherche
+Name[ga]=Breiseán Barra Cuardaigh
 Name[gl]=Plugin de Barra de Procura
 Name[he]=תוסף סרגל חיפוש
 Name[hi]=सर्च बार प्लगइन
@@ -42,7 +43,7 @@
 Name[nds]=Söökbalken-Moduul
 Name[nl]=Zoekbalkplugin
 Name[nn]=Søkelinjeprogramtillegg
-Name[pa]=ਖੋਜ ਪੱਟੀ ਪਲੱਗਿੰਨ
+Name[pa]=ਖੋਜ ਪੱਟੀ ਪਲੱਗਇਨ
 Name[pl]=Wtyczka paska wyszukiwania
 Name[pt]='Plugin' de Barra de Procura
 Name[pt_BR]=Plug-in de Barra de Busca
@@ -77,6 +78,7 @@
 Comment[gl]=Dá-lle unha caixa de texto para aceder directamente a motores de procura como Google.
 Comment[he]= מציג לך שורת קלט לגישה ישירה אל מנועי חיפוש כמו Google
 Comment[hi]=गूगल जैसे सर्च इंजनों पर सीधे पहुँच के लिए आपको एक पाठ बक्सा प्रदान करता है.
+Comment[hr]=Pruža tekstualni okvir za izravan pristup tražilicama poput Google-a.
 Comment[hu]=Gyors hozzáférést biztosít az internetes keresők (pl. Google) szolgáltatásaihoz.
 Comment[is]=Birtir textabox sem er beintengt leitarvélum eins og Google.
 Comment[it]=Ti presenta una casella di testo per l'accesso diretto a dei motori di ricerca quali Google.
Index: konq-plugins/arkplugin/arkplugin.desktop
===================================================================
--- konq-plugins/arkplugin/arkplugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/arkplugin/arkplugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -18,6 +18,7 @@
 Name[gl]=Comprimir/Extrair Ficheiros con Ark
 Name[he]=דחוס/פרוס קבצים עם Ark
 Name[hi]=आर्क के द्वारा फ़ाइलों को संपीडित करें/निकालें
+Name[hr]=Komprimiranje i izvlačenje datoteka pomoću aplikacije Ark
 Name[hu]=Fájlok tömörítése és kibontása az Arkkal
 Name[is]=Þjappa/taka út skrár með Ark
 Name[it]=Comprimi/estrai file con Ark
Index: konq-plugins/uachanger/uachangerplugin.desktop
===================================================================
--- konq-plugins/uachanger/uachangerplugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/uachanger/uachangerplugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -23,6 +23,7 @@
 Name[fa]=تغییردهندۀ عامل کاربر
 Name[fi]=Käyttäjäagenttimuuttaja
 Name[fr]=Changeur d'identité du navigateur
+Name[ga]=Clár Athraithe UserAgent
 Name[gl]=Cámbio de Identificación do Navegador
 Name[he]=משנה זיהוי דפדפן
 Name[hi]=यूज़रएजेंट परिवर्तक
@@ -71,6 +72,7 @@
 Comment[fa]=وصله برای تغییر عامل کاربر
 Comment[fi]=Sovelma käyttäjäagentin muuttamiseen
 Comment[fr]=Module externe pour changer l'identification du navigateur
+Comment[ga]=Breiseán a athraíonn an gníomhaire úsáideora
 Comment[gl]=Un plugin para alterar o axente do usuário
 Comment[he]=תוסף לשינוי זיהוי הדפדפן
 Comment[hi]=यूज़रएजेंट बदलने के लिए प्लगइन
Index: konq-plugins/webarchiver/webarchivecreator.cpp
===================================================================
--- konq-plugins/webarchiver/webarchivecreator.cpp	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/webarchiver/webarchivecreator.cpp	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -32,7 +32,7 @@
 
 extern "C"
 {
-	ThumbCreator *new_creator()
+	ThumbCreator * KDE_EXPORT new_creator()
 	{
 		return new WebArchiveCreator;
 	}
Index: konq-plugins/webarchiver/archivedialog.cpp
===================================================================
--- konq-plugins/webarchiver/archivedialog.cpp	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/webarchiver/archivedialog.cpp	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -395,13 +395,16 @@
 QString ArchiveDialog::handleLink(const KURL& _url, const QString& _link)
 {
    KURL url(getAbsoluteURL(_url, _link));
+
    QString tarFileName;
+   if (kapp->authorizeURLAction("redirect", _url, url))
+   {
+       if (m_state==Retrieving)
+           m_urlsToDownload.append(url);
+       else if (m_state==Saving)
+           tarFileName = m_downloadedURLDict[url.url()];
+   }
 
-   if (m_state==Retrieving)
-      m_urlsToDownload.append(url);
-   else if (m_state==Saving)
-      tarFileName = m_downloadedURLDict[url.url()];
-
    return tarFileName;
 }
 
Index: konq-plugins/webarchiver/plugin_webarchiver.desktop
===================================================================
--- konq-plugins/webarchiver/plugin_webarchiver.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/webarchiver/plugin_webarchiver.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -29,6 +29,7 @@
 Name[gl]=Arquivador Web
 Name[he]=ארכיוני רשת
 Name[hi]=वेब अभिलेखागार
+Name[hr]=Web-arhiviranje
 Name[hu]=Weboldal-archiváló
 Name[is]=Safnvistuð vefgögn
 Name[it]=Immagazzinatore Web
@@ -76,9 +77,11 @@
 Comment[fa]=بایگانیهای وب‌گاهها را ایجاد می‌کند
 Comment[fi]=Luo webbisivustojen arkistoja
 Comment[fr]=Crée des archives de sites Internet
+Comment[ga]=Cruthaigh cartlann de shuímh Ghréasáin
 Comment[gl]=Cria ficheiros de páxinas web
 Comment[he]=יצירת ארכיונים של אתרי אינטרנט
 Comment[hi]=वेबसाइटों के अभिलेखागार बनाए
+Comment[hr]=Izrada arhiva od web-lokacija
 Comment[hu]=Website-archiválás
 Comment[is]=Býr til safnskrár af vefsetrum
 Comment[it]=Crea archivi dei siti web
Index: konq-plugins/imagerotation/imageconverter.desktop
===================================================================
--- konq-plugins/imagerotation/imageconverter.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/imagerotation/imageconverter.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -18,6 +18,7 @@
 X-KDE-Submenu[fa]=تبدیل به
 X-KDE-Submenu[fi]=Muunna
 X-KDE-Submenu[fr]=Convertir en
+X-KDE-Submenu[ga]=Tiontaigh Go
 X-KDE-Submenu[gl]=Converter En
 X-KDE-Submenu[hr]=Pretvori u
 X-KDE-Submenu[hu]=Átalakítás
Index: konq-plugins/imagerotation/jpegorient.desktop
===================================================================
--- konq-plugins/imagerotation/jpegorient.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/imagerotation/jpegorient.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -14,7 +14,9 @@
 X-KDE-Submenu[fa]=تبدیل تصویر
 X-KDE-Submenu[fi]=Muuta kuva
 X-KDE-Submenu[fr]=Transformer l'image
+X-KDE-Submenu[ga]=Trasfhoirmigh Íomhá
 X-KDE-Submenu[gl]=Transformar a Imaxe
+X-KDE-Submenu[hr]=Pretvaranje slike
 X-KDE-Submenu[hu]=Képátalakítás
 X-KDE-Submenu[is]=Ummynda
 X-KDE-Submenu[it]=Trasforma immagine
@@ -232,7 +234,7 @@
 Name[es]=Reflejar horizontalmente
 Name[et]=Keera ümber horisontaalselt
 Name[eu]=Biratu horizontalki
-Name[fa]=قرینۀ عمودی
+Name[fa]=قرینۀ افقی
 Name[fi]=Käännä vaakasuorassa
 Name[fr]=Retourner horizontalement
 Name[ga]=Smeach go Cothrománach
Index: konq-plugins/kuick/kcmkuick/kcmkuick.desktop
===================================================================
--- konq-plugins/kuick/kcmkuick/kcmkuick.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/kuick/kcmkuick/kcmkuick.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -43,7 +43,7 @@
 Name[nl]=Snel kopiëren en verplaatsen
 Name[nn]=Snøggkopi-og-flytt-modul
 Name[nso]=Plugin ya Kapela ya Gatisa & Sutisa
-Name[pa]=ਚੁਸਤ ਨਕਲ ਅਤੇ ਭੇਜਣ ਪਲੱਗਿੰਨ
+Name[pa]=ਚੁਸਤ ਨਕਲ ਅਤੇ ਭੇਜਣ ਪਲੱਗਇਨ
 Name[pl]=Szybkie kopiowanie i przesuwanie
 Name[pt]='Plugin' de Cópia e Mudança Rápida
 Name[pt_BR]= Plugin de Copiar & Mover Rápido
Index: konq-plugins/kuick/kuick_plugin.desktop
===================================================================
--- konq-plugins/kuick/kuick_plugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/kuick/kuick_plugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -21,7 +21,7 @@
 Name[gl]=Copiar e Mover Rapidamente
 Name[he]=העתקה והעברה מהירה
 Name[hi]=क्विक नक़ल तथा खिसकाना
-Name[hr]=KBrzo kopiranje i premejštanje
+Name[hr]=KBrzo kopiranje i premještanje
 Name[hu]=Kuick - másolás és mozgatás
 Name[it]=Copia e spostamento rapidi
 Name[ja]=高速コピー & 移動
Index: konq-plugins/kuick/kuickplugin.desktop
===================================================================
--- konq-plugins/kuick/kuickplugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/kuick/kuickplugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -20,7 +20,7 @@
 Name[gl]=Copiar e Mover Rapidamente
 Name[he]=העתקה והעברה מהירה
 Name[hi]=क्विक नक़ल तथा खिसकाना
-Name[hr]=KBrzo kopiranje i premejštanje
+Name[hr]=KBrzo kopiranje i premještanje
 Name[hu]=Kuick - másolás és mozgatás
 Name[it]=Copia e spostamento rapidi
 Name[ja]=高速コピー & 移動
Index: konq-plugins/validators/plugin_validators.desktop
===================================================================
--- konq-plugins/validators/plugin_validators.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/validators/plugin_validators.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -23,9 +23,11 @@
 Name[fa]=اعتبارسنجهای وب‌گاه
 Name[fi]=WWW-validaattorit
 Name[fr]=Validateur de sites Internet
+Name[ga]=Bailíochtóirí Suímh Ghréasáin
 Name[gl]=Validación de Páxinas Web
 Name[he]=מוודאי אתרי אינטרנט
 Name[hi]=वेबसाइट वेलिडेटर्स
+Name[hr]=Provjera valjanosti web-lokacije
 Name[hu]=Weboldal-ellenőrző
 Name[is]=Villuleitun vefa
 Name[it]=Validatore di Siti Web
@@ -73,9 +75,11 @@
 Comment[fa]=وصله‌های اعتبارسنج زنگام و CSS
 Comment[fi]=CSS ja HTML validointisovelmat
 Comment[fr]=Module de validation CSS et HTML
+Comment[ga]=Breiseáin Bhailíochtaithe CSS agus HTML
 Comment[gl]=Plugins de validación de CSS e HTML 
 Comment[he]=תוספים לווידוא CSS ו־HTML
 Comment[hi]=सीएसएस तथा एचटीएमएल वेलिडेटर प्लगइन्स
+Comment[hr]=Dodatak za provjeravanje valjanosti CSS-a i HTML-a
 Comment[hu]=CSS- és HTML-ellenőrző bővítőmodul
 Comment[is]=CSS og HTML gildir
 Comment[it]=Plugin per validare CSS e HTML
Index: konq-plugins/sidebar/metabar/src/metabar.desktop
===================================================================
--- konq-plugins/sidebar/metabar/src/metabar.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/sidebar/metabar/src/metabar.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -17,7 +17,9 @@
 Comment[fa]=وصلۀ تابلوی ناوش konqueror که فرامیله نام دارد
 Comment[fi]=Konquerorin Metabar-niminen selauspaneelin liitännäinen
 Comment[fr]=Un module pour le panneau de navigation de Konqueror appelé Metabar
+Comment[ga]=Breiseán i bpainéal nascleanúna Konqueror darb ainm Metabar
 Comment[gl]=Un plugin de navegación para konqueror chamado Metabar
+Comment[hr]=Dodatak navigacijske ploče za Konqueror, zvan Metabar
 Comment[hu]=Metabar, egy navigációs bővítőmodul a Konqueror böngészőhöz
 Comment[is]=Konqueror leiðarstýrispjald íforrit sem kallast Metabar
 Comment[it]=Plugin di navigazione di Konqueror chiamato Metabar
@@ -50,7 +52,6 @@
 Name[fa]=فرامیله
 Name[fi]=metapalkki
 Name[fr]=Metabar
-Name[ga]=meiteabharra
 Name[hu]=Metabar
 Name[ja]=メタバー
 Name[km]=របារ​មេតា
Index: konq-plugins/sidebar/metabar/src/metabar_add.desktop
===================================================================
--- konq-plugins/sidebar/metabar/src/metabar_add.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/sidebar/metabar/src/metabar_add.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -13,6 +13,7 @@
 Comment[fa]=وصلۀ تابلوی ناوش konqueror که فرامیله نام دارد
 Comment[fi]=Konquerorin Metabar -niminen selauspaneelin liitännäinen
 Comment[fr]=Un module pour le panneau de navigation de Konqueror appelé Metabar
+Comment[ga]=Breiseán i bpainéal nascleanúna Konqueror darb ainm Metabar
 Comment[gl]=Un plugin de navegación para konqueror chamado Metabar
 Comment[hr]=Dodatak navigacijske trake za Konqueror, nazvan Metabar
 Comment[hu]=Metabar, egy navigációs bővítőmodul a Konqueror böngészőhöz
@@ -45,7 +46,6 @@
 Name[el]=Γραμμή μεταδεδομένων
 Name[es]=Metabarra
 Name[fa]=فرامیله
-Name[ga]=Meiteabharra
 Name[ja]=メタバー
 Name[km]=របារ​មេតា
 Name[lt]=Meta juosta
Index: konq-plugins/sidebar/mediaplayer/mediaplayerplugin.desktop
===================================================================
--- konq-plugins/sidebar/mediaplayer/mediaplayerplugin.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/sidebar/mediaplayer/mediaplayerplugin.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -62,6 +62,7 @@
 Comment[fa]=وصلۀ پخش‌کنندۀ رسانه در تابلوی ناوش Konqueror
 Comment[fi]=Mediasoitinsovelma Konquerorin navigaatiopaneeliin
 Comment[fr]=Le module de lecteur multimédia pour la barre de navigation de Konqueror
+Comment[ga]=Breiseán Mediaplayer i bpainéal nascleanúna Konqueror
 Comment[gl]=Un plugin de reproduición multimédia no painel de navegación de Konqueror
 Comment[he]=תוסף נגן מדיה ללוח הניווט של Konqueror
 Comment[hi]=कॉन्करर के नेविगेशन फलक में मीडियाप्लेयर प्लगइन
Index: konq-plugins/sidebar/newsticker/news_add.desktop
===================================================================
--- konq-plugins/sidebar/newsticker/news_add.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ konq-plugins/sidebar/newsticker/news_add.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -14,7 +14,7 @@
 Name[gl]=Notícias
 Name[he]=מציג חדשות
 Name[hi]=न्यूज़टिकर
-Name[hr]=Obavijest o novostima
+Name[hr]=Obavijesti o novostima
 Name[hu]=Hírmegjelenítő
 Name[is]=Fréttastrimill
 Name[it]=NewsTicker
Index: atlantikdesigner/atlantikdesigner.desktop
===================================================================
--- atlantikdesigner/atlantikdesigner.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ atlantikdesigner/atlantikdesigner.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -65,10 +65,11 @@
 Comment[es]=Diseñador de tableros de juego para Atlantik
 Comment[et]=Atlantik mängulaua disainer
 Comment[eu]=Atlantik jokorako taula diseinatzeko tresna
-Comment[fa]=طراح تختۀ بازی برای Atlantik
+Comment[fa]=طراح تخته بازی برای Atlantik
 Comment[fi]=Pelilaudan suunnitteluohjelma Atlantikiin
 Comment[fo]=Spælabrettissniðari til Atlantik
 Comment[fr]=Concepteur de plateaux de jeu pour Atlantik
+Comment[ga]=Dearthóir clár cluiche le haghaidh Atlantik
 Comment[gl]=O deseñador de taboleiro de Atlantik
 Comment[he]=מעצב לוחות משחק ל־Atlantik
 Comment[hi]=अटलांटिक के लिए खेल बिसातडिज़ाइनर
@@ -121,6 +122,7 @@
 GenericName[eu]=Monopoly®-bezalako diseinatzailea
 GenericName[fi]=Monopoly®-tyylisen pelin suunnitteluohjelma
 GenericName[fr]=Concepteur de plateaux dans le style du Monopoly®
+GenericName[ga]=Dearthóir Cluichí cosúil le Monopoly®
 GenericName[gl]=Deseñador de Taboleiros tipo Monopoly®
 GenericName[he]=בונה לוחות לתואמי מונופל®
 GenericName[hr]=Dizajner ploče za igre poput Monopola®
Index: kaddressbook-plugins/xxports/gmx/gmx_xxport.desktop
===================================================================
--- kaddressbook-plugins/xxports/gmx/gmx_xxport.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kaddressbook-plugins/xxports/gmx/gmx_xxport.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -29,7 +29,7 @@
 Name[nds]=GMX-Moduul för KAdressbook
 Name[nl]=KAB GMX XXPort-plugin
 Name[nn]=Programtillegg KAB GMX XXPort
-Name[pa]=KAB GMX XXPort ਪਲੱਗਿੰਨ
+Name[pa]=KAB GMX XXPort ਪਲੱਗਇਨ
 Name[pl]=Wtyczka KAB GMX XXPort
 Name[pt]='Plugin' XXPort GMX do KAB
 Name[pt_BR]=Plug-in GMX para o Livro de Endereços
@@ -59,9 +59,11 @@
 Comment[fa]=وصله برای واردات و صادرات تماسها در قالب کتاب نشانی GMX
 Comment[fi]=Sovelma GMX-osoitekirjamuodossa olevien tietueiden tuontiin ja vientiin
 Comment[fr]=Module externe permettant d'importer et d'exporter des contacts avec le format du carnet d'adresses GMX
+Comment[ga]=Breiseán chun teagmhálacha i bhformáid leabhair seoltaí GMX a iompórtáil agus a easpórtáil
 Comment[gl]=Un plugin para importar e exportar contactos no formato de libro de enderezos GMX
 Comment[he]= תוסף לייבוא וייצוא של אנשי קשר בפורמט פנקס הכתובות של GMX
 Comment[hi]=जीएमएक्स पता पुस्तिका फार्मेट में सम्पर्कों को आयात और निर्यात करने का प्लगइन
+Comment[hr]=Dodatak za uvoz i izvoz kontakata u oblikovanju GMX adresara
 Comment[hu]=Bővítőmodul GMX formátumú névjegyek importálásához és exportálásához
 Comment[is]=Íforrit sem flytur inn og út tengiliði í GMX vistfangasniði
 Comment[it]=Plugin per importare ed esportare contatti in formato GXM
Index: kaddressbook-plugins/xxports/kworldclock/geo_xxport.desktop
===================================================================
--- kaddressbook-plugins/xxports/kworldclock/geo_xxport.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kaddressbook-plugins/xxports/kworldclock/geo_xxport.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -30,7 +30,7 @@
 Name[nds]=KWorldClock-Moduul för KAdressbook
 Name[nl]=KAB KWorldClock XXPort-plugin
 Name[nn]=Programtillegg KAB KWorldClock XXPort
-Name[pa]=KAB KWorldClock XXPort ਪਲੱਗਿੰਨ
+Name[pa]=KAB KWorldClock XXPort ਪਲੱਗਇਨ
 Name[pl]=Wtyczka eksportu danych KAB do KWorldClock
 Name[pt]='Plugin' de XXPort do KWorldClock para o KAB
 Name[pt_BR]=Plug-Ins do KWorldClock do Kab
@@ -61,11 +61,13 @@
 Comment[es]=Extensión para exportar datos geográficos de los contactos como banderas en KWorldClock
 Comment[et]=Plugin, mis ekspordib KWorldClocki aadressiraamatus leiduvate isikute asukoha lipukestena
 Comment[eu]=KWorldClock-eko kontaktuen datu geografikoak bandera gisa esportatzeko plugina
-Comment[fa]=وصله برای صادرات دادۀ جغرافیای تماسها به عنوان پرچمهای KWorldClock
+Comment[fa]=وصله برای صادرات داده‌های جغرافیایی تماسها به عنوان پرچمهای KWorldClock
 Comment[fi]=Liitännäinen maantieteellisten koordinaattien siirtämiseen KWorldClock sovellukseen
 Comment[fr]=Module externe pour exporter les données géographiques des contacts sous forme de drapeaux dans KWorldClock
+Comment[ga]=Breiseán a easpórtálann na sonraí geografacha de theagmhálacha mar bhratacha i KWorldClock
 Comment[gl]=Un plugin para exportar os dados xeográficos dos contactos como bandeiras en KWorldClock
 Comment[hi]=कान्टेक्ट के भौगोलिक डाटा को के-वर्ल्डक्लॉक में फ्लेग्स की तरह निर्यात करने का प्लगइन
+Comment[hr]=Dodatak za izvoz geografskih podataka kontakata u obliku zastava u KWorldClock
 Comment[hu]=Bővítőmodul a névjegyek földrajzi adatainak KWorldClock-ba való exportálásához (jelzők formájában)
 Comment[is]=Íforrit sem flytur út landupplýsingar um staðsetningar fyrir KWorldClock
 Comment[it]=Plugin per esportare geo data dei contatti come bandiere in KWorldClock
Index: ksig/ksig.desktop
===================================================================
--- ksig/ksig.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ ksig/ksig.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -30,6 +30,7 @@
 GenericName[gl]=Editor de Sinaturas
 GenericName[he]=עורך חתימות
 GenericName[hi]=हस्ताक्षर संपादक
+GenericName[hr]=Uređivač potpisa
 GenericName[hu]=Aláírásszerkesztő
 GenericName[is]=Ritill fyrir undirritanir
 GenericName[it]=Editor della firma
Index: renamedlgplugins/images/renimagedlg.desktop
===================================================================
--- renamedlgplugins/images/renimagedlg.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ renamedlgplugins/images/renimagedlg.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -18,6 +18,7 @@
 Name[fi]=Kuvannäyttäjä
 Name[fo]=Myndaframvísari
 Name[fr]=Afficheur d'images
+Name[ga]=Amharcán Íomhánna
 Name[gl]=Visor de Imaxes
 Name[he]=מציג תמונות
 Name[hi]=छवि प्रदर्शक
Index: kfile-plugins/lnk/lnkforward.desktop
===================================================================
--- kfile-plugins/lnk/lnkforward.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kfile-plugins/lnk/lnkforward.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -15,9 +15,11 @@
 Comment[fa]=آغاز پیوند داده‌شده در پروندۀ .lnk ویندوز
 Comment[fi]=Käynnistä Windows .lnk-tiedostossa annettu linkki
 Comment[fr]=Lancer le lien issu d'un fichier « .lnk » Windows
+Comment[ga]=Nasc tosaigh ón chomhad .lnk Windows
 Comment[gl]=Iniciar o enlace dado nun ficheiro .lnk de Windows
 Comment[he]=התחל קיצור בקובץ lnk של Windows
 Comment[hi]=विंडोज़ के  .lnk फ़ाइल में स्टार्ट लिंक दिया गया है
+Comment[hr]=Pokretanje veze iz Windows datoteke .lnk
 Comment[hu]=Link megnyitása Windows-os Lnk-fájl alapján
 Comment[is]=Ræsa tengil í Windows lnk skrá
 Comment[it]=Collegamento iniziale dato nel file .lnk di Windows
Index: kfile-plugins/lnk/kfile_lnk.desktop
===================================================================
--- kfile-plugins/lnk/kfile_lnk.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kfile-plugins/lnk/kfile_lnk.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -16,6 +16,7 @@
 Name[fa]=اطلاعات پروندۀ lnk ویندوز
 Name[fi]=Windows lnk-tiedoston tiedot
 Name[fr]=Informations sur le fichier « .lnk » Windows
+Name[ga]=Eolas faoin Chomhad lnk Windows
 Name[gl]=Información de Ficheiros lnk de Windows
 Name[he]= מידע קובץ lnk של Windows
 Name[hi]=विंडो लिंक फ़ाइल जानकारी
Index: kfile-plugins/lnk/x-win-lnk.desktop
===================================================================
--- kfile-plugins/lnk/x-win-lnk.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kfile-plugins/lnk/x-win-lnk.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -15,6 +15,7 @@
 Comment[fa]=پروندۀ پیوند ویندوز
 Comment[fi]=Windows linkkitiedosto
 Comment[fr]=Fichier de lien de Windows
+Comment[ga]=Comhad Naisc Windows
 Comment[gl]=Ficheiro Windows Link
 Comment[he]=קיצור דרך של Windows
 Comment[hi]=विंडो लिंक-फ़ाइल
Index: kfile-plugins/cert/kfile_cert.desktop
===================================================================
--- kfile-plugins/cert/kfile_cert.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kfile-plugins/cert/kfile_cert.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -15,6 +15,7 @@
 Name[fa]=اطلاعات گواهی‌نامه
 Name[fi]=Sertifikaatin tiedot
 Name[fr]=Informations sur le certificat
+Name[ga]=Eolas faoin Teastas
 Name[gl]=Informacións do Certificado
 Name[hi]=प्रमाणपत्र जानकारी
 Name[hr]=Podaci potvrde
Index: kicker-applets/mediacontrol/mediacontrol.desktop
===================================================================
--- kicker-applets/mediacontrol/mediacontrol.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kicker-applets/mediacontrol/mediacontrol.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -17,7 +17,7 @@
 Name[fr]=Télécommande multimédia
 Name[gl]=Controlo Multimédia
 Name[he]=בקרת מדיה
-Name[hr]=Upravaljanje medijima
+Name[hr]=Upravljanje medijima
 Name[hu]=Médiavezérlő
 Name[is]=Margmiðlunarstjórn
 Name[it]=Controllo multimediale
@@ -28,6 +28,7 @@
 Name[nb]=Mediastyring
 Name[nds]=Medienkuntrull
 Name[nn]=Mediekontroll
+Name[pa]=ਮੀਡਿਆ ਕੰਟਰੋਲ
 Name[pl]=Zarządzanie mediami
 Name[pt]=Controlo Multimédia
 Name[pt_BR]=Controle de Mídia
Index: kicker-applets/ktimemon/ktimemon.desktop
===================================================================
--- kicker-applets/ktimemon/ktimemon.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kicker-applets/ktimemon/ktimemon.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -34,6 +34,7 @@
 Comment[nds]=En eenfach Systeemkieker
 Comment[nl]=Een eenvoudige systeemmonitor
 Comment[nn]=Ein enkel systemovervakar
+Comment[pa]=ਇੱਕ ਸਧਾਰਨ ਸਿਸਟਮ ਨਿਗਰਾਨ
 Comment[pl]=Prosty monitor systemu
 Comment[pt]=Um monitor do sistema simples
 Comment[pt_BR]=Um monitor de sistema simples
Index: kicker-applets/kolourpicker/kolourpicker.desktop
===================================================================
--- kicker-applets/kolourpicker/kolourpicker.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kicker-applets/kolourpicker/kolourpicker.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -87,6 +87,7 @@
 Comment[fi]=Valitse värejä kaikkialta ja tutki värien arvoja
 Comment[fo]=Vel litir frá skriviborði.
 Comment[fr]=Capture une couleur n'importe où et obtient sa composition
+Comment[ga]=Roghnaigh dath ó áit ar bith agus faigh a luach
 Comment[gl]=Extrai as cores de calquer sítio e obtén os seus valores
 Comment[he]=בחירת צבעים מכל מקום שהוא וקבלת ערכיהם
 Comment[hi]=कहीं से भी रंग चुने तथा उसका मान प्रदर्शित करता है
Index: kicker-applets/math/mathapplet.desktop
===================================================================
--- kicker-applets/math/mathapplet.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kicker-applets/math/mathapplet.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -20,7 +20,7 @@
 Name[gl]=Avaliación de Expresións Matemáticas
 Name[he]=מפענח נוסחאות מתמטיות
 Name[hi]=मैथ एक्सप्रेशन इवेल्यूएटर्स
-Name[hr]=Procjena matematičkih izraza
+Name[hr]=Ocjena matematičkih izraza
 Name[hu]=Kifejezéskiértékelő
 Name[is]=Algebrureiknivél
 Name[it]=Disegnatore di funzioni matematiche
@@ -68,7 +68,7 @@
 Comment[gl]=Un avaliador de expresións matemáticas
 Comment[he]=מפענח נוסחאות מתמטיות
 Comment[hi]=एक मैथमेटिकल एक्सप्रेशन इवेल्यूएटर
-Comment[hr]=Procjenjivanje matematičkih izraza
+Comment[hr]=Ocjenjivanje matematičkih izraza
 Comment[hu]=Matematikai kifejezések kiszámítására alkalmas program
 Comment[is]=Algebrureiknivél
 Comment[it]=Uno strumento per disegnare funzioni matematiche
Index: kicker-applets/kbinaryclock/kbinaryclock.desktop
===================================================================
--- kicker-applets/kbinaryclock/kbinaryclock.desktop	(.../tags/KDE/3.5.6/kdeaddons)	(wersja 627683)
+++ kicker-applets/kbinaryclock/kbinaryclock.desktop	(.../branches/KDE/3.5/kdeaddons)	(wersja 627683)
@@ -63,6 +63,7 @@
 Comment[fa]=فقره‌ها را در قالب دودویی نمایش می‌دهد
 Comment[fi]=Näyttää ajan binäärimuodossa
 Comment[fr]=Affiche l'heure en binaire
+Comment[ga]=Taispeáin an t-am i bhformáid dhénártha
 Comment[gl]=Mostra a hora en formato binário
 Comment[hr]=Prikaz vremena u binarnom obliku
 Comment[hu]=Bináris formátumban jeleníti meg az időt

Zmiany atrybutów dla: .
___________________________________________________________________
Nazwa: svn:externals
   + admin https://svn.kde.org/home/kde/branches/KDE/3.5/kde-common/admin


