diff -urN -x CVS kdeaddons.orig/atlantikdesigner/atlantikdesigner.desktop kdeaddons/atlantikdesigner/atlantikdesigner.desktop
--- kdeaddons.orig/atlantikdesigner/atlantikdesigner.desktop	Fri Mar 12 13:41:01 2004
+++ kdeaddons/atlantikdesigner/atlantikdesigner.desktop	Sun Apr 25 00:50:51 2004
@@ -7,7 +7,7 @@
 MapNotify=true 
 Name=Atlantik Designer
 Name[af]=Atlantik Ontwerper
-Name[ar]=مصمم Atlantik
+Name[ar]=مصمم أتلانتيك
 Name[ca]=Dissenyador Atlantik
 Name[cs]=Atlantik designer
 Name[cy]=Dylunydd Atlantik
@@ -68,11 +68,11 @@
 Comment[ms]=Pereka permainan papan untuk Atlantik
 Comment[mt]=Diżinjatur ta' logħob tal-bord għall-Atlantik
 Comment[nb]=Spillebrettbygger for Atlantik
-Comment[nl]=Spelbord-ontwerper voor Atlantik
+Comment[nl]=Spelbordontwerper voor Atlantik
 Comment[nn]=Utforming av spelebrett for Atlantik
 Comment[pl]=Program do projektowania plansz dla programu Atlantyk
 Comment[pt]=O desenhador do tabuleiro do Atlantik
-Comment[pt_BR]=O designer de jogo para  o Atlantik
+Comment[pt_BR]=O tabuleiro para o Atlantik Designer
 Comment[ro]=Desenator de table de joc pentru Atlantik
 Comment[ru]=Разработка дизайна для игры Atlantik
 Comment[sk]=Návrhár pre hru Atlantik
diff -urN -x CVS kdeaddons.orig/atlantikdesigner/designer/designer.cpp kdeaddons/atlantikdesigner/designer/designer.cpp
--- kdeaddons.orig/atlantikdesigner/designer/designer.cpp	Fri Jan 31 06:43:02 2003
+++ kdeaddons/atlantikdesigner/designer/designer.cpp	Wed Apr 14 13:02:21 2004
@@ -157,7 +157,7 @@
 		m_player = new Player(0);
 	else
 		m_player->setLocation(0);
-	
+
 	board->addToken(m_player);
 
 	connect(m_player, SIGNAL(changed(Player *)), board, SLOT(playerChanged(Player *)));
@@ -211,7 +211,7 @@
 {
 	if (isMod)
 	{
-		int result = KMessageBox::warningYesNoCancel(this, i18n("There are unsaved changes to gameboard. Save them?"), i18n("Unsaved Changes"), i18n("Save"), i18n("Discard"), "DiscardAsk", true);
+		int result = KMessageBox::warningYesNoCancel(this, i18n("There are unsaved changes to gameboard. Save them?"), i18n("Unsaved Changes"), KStdGuiItem::save(), KStdGuiItem::discard(), "DiscardAsk", true);
 		switch(result)
 		{
 		case KMessageBox::Yes:
@@ -511,7 +511,7 @@
 					break;
 				}
 			}
-			
+
 			if (found)
 				continue;
 
@@ -698,7 +698,7 @@
 	}
 
 	// now do the cards
-	
+
 	for (QValueList<CardStack>::Iterator it = cards.begin(); it != cards.end(); ++it)
 	{
 		t << endl << endl;
@@ -737,7 +737,7 @@
 				{
 					t << "canbeowned=" << 1 << endl;
 				}
-				
+
 				t << key << "=" << value << endl;
 			}
 		}
diff -urN -x CVS kdeaddons.orig/debian/changelog kdeaddons/debian/changelog
--- kdeaddons.orig/debian/changelog	Sat Mar 13 00:20:44 2004
+++ kdeaddons/debian/changelog	Sat Apr 24 00:54:24 2004
@@ -1,3 +1,9 @@
+kdeaddons (4:3.2.2-1) unstable; urgency=low
+
+  * New upstream bugfix release.
+
+ -- Ben Burton <bab@debian.org>  Sat, 24 Apr 2004 08:53:43 +1000
+
 kdeaddons (4:3.2.1-1) unstable; urgency=low
 
   * First KDE 3.2 upload to sid.
diff -urN -x CVS kdeaddons.orig/kaddressbook-plugins/xxports/kworldclock/geo_xxport.desktop kdeaddons/kaddressbook-plugins/xxports/kworldclock/geo_xxport.desktop
--- kdeaddons.orig/kaddressbook-plugins/xxports/kworldclock/geo_xxport.desktop	Sun Feb 29 15:40:36 2004
+++ kdeaddons/kaddressbook-plugins/xxports/kworldclock/geo_xxport.desktop	Sun Apr 18 14:31:13 2004
@@ -19,7 +19,7 @@
 Name[nn]=Programtillegg KAB KWorldClock XXPort
 Name[pl]=Wtyczka eksportu danych KAB do KWorldClock
 Name[pt]='Plugin' de XXPort do KWorldClock para o KAB
-Name[pt_BR]=Plug-Ins do KWorldClock do Kab
+Name[pt_BR]=Plug-in do KWorldClock do Kab
 Name[ro]=Modul de exportare KAB-KWorldClock
 Name[ru]=Модуль KAB KWorldClock XXPort
 Name[sk]=Module KWorldClock pre KAB
@@ -49,11 +49,12 @@
 Comment[it]=Plugin per esportare geo data dei contatti come bandiere in KWorldClock
 Comment[ja]=KWorldClockの旗を地図データの接点にエクスポートするプラグイン
 Comment[ms]=Plug masuk intuk eksport data geo hubungan sebagai tanda dalam KWorldClock
+Comment[nb]=Modul som eksporterer geografiske data om kontakter som flagg i KWorldClock
 Comment[nl]=Plugin voor het exporteren van de geografische gegevens van contacten als vlaggen in KworldClock
 Comment[nn]=Programtillegg for å eksportera geografiske data frå kontaktar til flagg i KWorldClock
 Comment[pl]=Wtyczka do eksportu danych geograficznych z wizytówek jako flag w KWorldClock
 Comment[pt]=Um 'plugin' para exportar os dados geográficos dos contactos como bandeiras no KWorldClock
-Comment[pt_BR]=Plugin para exportar dados geográficos de contatos, como sinais no Kworldclock
+Comment[pt_BR]=Plug-in para exportar dados geográficos de contatos, como sinais no Kworldclock
 Comment[ro]=Modul care exportă datele geografice din adresele de contact ca fanioane în KWorldClock
 Comment[ru]=Модуль экспорта географических данных контактов в виде флагов KWorldClock
 Comment[sk]=Modul pre export geografických dát kontaktov ako vlajok pre KWorldClock
diff -urN -x CVS kdeaddons.orig/kate/cppsymbolviewer/katecppsymbolviewer.desktop kdeaddons/kate/cppsymbolviewer/katecppsymbolviewer.desktop
--- kdeaddons.orig/kate/cppsymbolviewer/katecppsymbolviewer.desktop	Fri Mar 12 13:41:03 2004
+++ kdeaddons/kate/cppsymbolviewer/katecppsymbolviewer.desktop	Sun Apr 25 00:50:53 2004
@@ -4,6 +4,7 @@
 X-KDE-Library=katecppsymbolviewerplugin
 X-Kate-Version=2.2
 Name=Kate C/C++ symbol viewer
+Name[ar]=مستعرض علامات C/C++ لكيت
 Name[ca]=Visor de símbols C/C++ per a Kate
 Name[cs]=Prohlížeč symbolů C/C++pro Kate
 Name[cy]=Gwelydd symbolau C/C++ Kate
@@ -22,6 +23,7 @@
 Name[it]=Visualizzatore Kate per simboli C/C++
 Name[ja]=Kate C/C++ シンボル ビューア
 Name[ms]=Pelihat simbol Kate C/C++
+Name[nb]=Kate C/C++ symbolviser
 Name[nl]=Kate C/C++ symbolen-weergaveprogramma
 Name[nn]=Kate C/C++-symbolvisar
 Name[pl]=Przeglądarka symboli C/C++ dla Kate
@@ -39,6 +41,7 @@
 Name[xx]=xxKate C/C++ symbol viewerxx
 Name[zh_CN]=Kate C/C++ 符号查看器
 Comment=This plugin extracts and shows C/C++ symbols from source
+Comment[ar]=هذا الملحق يستخرج و يعرض علامات C/C++ من الشفرة المصدرية
 Comment[bg]=Приставка за извличане и показване на символите C/C++ от изходния код
 Comment[ca]=Aquest connector extrau i mostra els símbols C/C++ des de la font
 Comment[cs]=Tento modul extrahuje a ukazuje C/C++ symboly ze zdroje
@@ -58,11 +61,12 @@
 Comment[it]=Questo plugin estrae e mostra simboli C/C++ dal sorgente
 Comment[ja]=このプラグインはソースからC/C++のシンボルを抽出して表示します
 Comment[ms]=Plug masuk ekstrak dan papar simbol C/C++ dari sumber
+Comment[nb]=Denne modulen finner og viser C/C++-symboler fra kildekoden
 Comment[nl]=Deze plugin haalt de C/C++-symbolen uit de broncode en toont deze
 Comment[nn]=Dette programtillegget hentar ut og viser C/C++-symbol frå kjeldekode
 Comment[pl]=Ta wtyczka pokazuje symbole C/C++ znalezione w pliku źródłowym
 Comment[pt]=Este 'plugin' extrai e mostra símbolos C/C++ do código fonte
-Comment[pt_BR]=Este plugin extrai e exibe símbolos C/C++ do código-fonte
+Comment[pt_BR]=Este plug-in extrai e exibe símbolos C/C++ do código-fonte
 Comment[ro]=Acest modul extrage şi afişează simbolurile C/C++ din fişierele sursă
 Comment[ru]=Извлечение и отображение идентификаторов C/C++ из исходного кода
 Comment[sk]=Tento modul získa a zobrazí symboly C/C++ zo zdrojového kódu
diff -urN -x CVS kdeaddons.orig/kate/filelistloader/katefll.desktop kdeaddons/kate/filelistloader/katefll.desktop
--- kdeaddons.orig/kate/filelistloader/katefll.desktop	Sun Mar  7 14:17:51 2004
+++ kdeaddons/kate/filelistloader/katefll.desktop	Sun Apr 25 00:50:55 2004
@@ -26,7 +26,7 @@
 GenericName[ms]=Penyunting Teks /FLL
 GenericName[mt]=Editur tat-test /FLL
 GenericName[nb]=Tekstredigerer / Fillaster
-GenericName[nl]=Texteditor /FLL
+GenericName[nl]=Teksteditor /FLL
 GenericName[nn]=Skriveprogram/FLL
 GenericName[nso]=/FLL ya Mofetosi wa Sengwalwana
 GenericName[pl]=Edytor tekstowy /Wczytywanie listy plików
diff -urN -x CVS kdeaddons.orig/kate/filelistloader/katefll_initplugin.desktop kdeaddons/kate/filelistloader/katefll_initplugin.desktop
--- kdeaddons.orig/kate/filelistloader/katefll_initplugin.desktop	Fri Mar 12 13:41:05 2004
+++ kdeaddons/kate/filelistloader/katefll_initplugin.desktop	Sun Apr 18 14:31:17 2004
@@ -28,7 +28,7 @@
 Name[nso]=Tsenyo ya init ya FLL ya Kate
 Name[pl]=Wtyczka wczytywania listy plików do Kate
 Name[pt]='Plugin' da Lista de Ficheiros do Kate
-Name[pt_BR]=Plugin FLL do Kate
+Name[pt_BR]=Plug-in FLL do Kate
 Name[ro]=Modul de iniţializare FFL pentru Kate
 Name[ru]=Модуль инициализации Kate FLL
 Name[sk]=Kate FLL inicializačný modul
diff -urN -x CVS kdeaddons.orig/kate/filelistloader/katefll_plugin.desktop kdeaddons/kate/filelistloader/katefll_plugin.desktop
--- kdeaddons.orig/kate/filelistloader/katefll_plugin.desktop	Sun Feb 29 15:40:44 2004
+++ kdeaddons/kate/filelistloader/katefll_plugin.desktop	Sun Apr 25 00:50:57 2004
@@ -16,7 +16,7 @@
 Name[fi]=Kate tiedostolistalatain
 Name[fr]=Chargeur de liste de fichiers pour Kate
 Name[he]=טוען רשימות הקבצים של Kate
-Name[hi]=केएटीई फाइल सूची लोड करने वाला
+Name[hi]=केएटीई फ़ाइल सूची लोड करने वाला
 Name[hu]=Kate fájllista-betöltő
 Name[is]=Kate skráarlistar
 Name[it]=Caricatore di Liste di File (FLL) di Kate
@@ -57,7 +57,7 @@
 Comment[fi]=Tallenna Katen tiedostolista levylle
 Comment[fr]=Enregistrement de la liste des fichiers de Kate sur le disque
 Comment[he]=שמירת רשימת הקבצים של Kate על גבי הכונן
-Comment[hi]=डिस्क पर केएटीई की फाइल सूची भंडारित करता है
+Comment[hi]=डिस्क पर केएटीई की फ़ाइल सूची भंडारित करता है
 Comment[hu]=A Kate fájllista elmentése lemezre
 Comment[is]=Geyma skráarlista Kate á diski
 Comment[it]=Salva la lista di file di Kate su disco
diff -urN -x CVS kdeaddons.orig/kate/helloworld/katehelloworld.desktop kdeaddons/kate/helloworld/katehelloworld.desktop
--- kdeaddons.orig/kate/helloworld/katehelloworld.desktop	Fri Apr  2 14:08:36 2004
+++ kdeaddons/kate/helloworld/katehelloworld.desktop	Sun Apr 25 00:50:59 2004
@@ -5,6 +5,7 @@
 X-Kate-Version=2.2
 Name=My first Kate Plugin
 Name[af]=My eerste Kate Inplak
+Name[ar]=ملحق Kate اﻷول
 Name[bs]=Moj prvi Kate dodatak
 Name[ca]=El meu primer connector per a Kate
 Name[cs]=Můj první modul pro Kate
@@ -31,7 +32,7 @@
 Name[nso]=Tsenyo yaka ya mathomo ya Kate
 Name[pl]=Moja pierwsza wtyczka do Kate
 Name[pt]=O meu primeiro 'plugin' do Kate
-Name[pt_BR]=Meu primeiro plugin do Kate
+Name[pt_BR]=Meu primeiro plug-in do Kate
 Name[ro]=Primul meu modul Kate
 Name[ru]=Мой первый модуль к Kate
 Name[sk]=Môj prvý modul pre Kate
@@ -49,6 +50,7 @@
 Name[zh_CN]=我的第一个 Kate 插件
 Comment=Your short description about the plugin goes here
 Comment[af]=Jou kort beskrywing aangaande die inplak gaan hier
+Comment[ar]=اكتب وصف قصير عن هذا الملحق هنا
 Comment[bg]=Описание на Вашата първа приставка за Kate
 Comment[bs]=Ovdje treba biti vaše malo objašnjenje o dodatku
 Comment[ca]=La vostra descripció referent al connector va aquí
@@ -76,7 +78,7 @@
 Comment[nso]=Thlaloso ya gago ye kopana kaga plugin e tsamaya mo
 Comment[pl]=Tu powinien być Twój krótki opis wtyczki
 Comment[pt]=Aqui deverá colocar uma descrição do 'plugin'
-Comment[pt_BR]=Uma pequena descrição maior sobre o que um plugin faz
+Comment[pt_BR]=Uma pequena descrição maior sobre o que um plug-in faz
 Comment[ro]=Aici trebuie să fie descrierea modulului dumneavoastră
 Comment[ru]=А здесь должно быть описание модуля
 Comment[sk]=Tu by mal byť popis o module
diff -urN -x CVS kdeaddons.orig/kate/htmltools/katehtmltools.desktop kdeaddons/kate/htmltools/katehtmltools.desktop
--- kdeaddons.orig/kate/htmltools/katehtmltools.desktop	Sun Feb 29 15:40:48 2004
+++ kdeaddons/kate/htmltools/katehtmltools.desktop	Sun Apr 25 00:51:01 2004
@@ -5,6 +5,7 @@
 X-Kate-Version=2.2
 Name=Kate HTML Tools
 Name[af]=Kate Html Nutsprogramme
+Name[ar]=أدوات Kate HTML
 Name[az]=Kate HTML Vasitələri
 Name[bs]=Kate HTML alati
 Name[ca]=Eines HTML per a Kate
@@ -58,6 +59,7 @@
 Name[zu]=Amathuluzi e Kate HTML.
 Comment=Facilitated insertion of HTML tags
 Comment[af]=Bystand invoeg van Html etiket
+Comment[ar]=ادخال أسهل لعلامات HTML
 Comment[az]=HTML təqlərinin səliqəli əlavəsi
 Comment[bg]=Помощ при вмъкване на тагове и елементи HTML
 Comment[bs]=Primjenjeno ubacivanje HTML tagova
diff -urN -x CVS kdeaddons.orig/kate/insertcommand/kateinsertcommand.desktop kdeaddons/kate/insertcommand/kateinsertcommand.desktop
--- kdeaddons.orig/kate/insertcommand/kateinsertcommand.desktop	Sun Feb 29 15:40:50 2004
+++ kdeaddons/kate/insertcommand/kateinsertcommand.desktop	Sun Apr 25 00:51:03 2004
@@ -45,6 +45,7 @@
 Name[zu]=Faka Inkomba ka Kate
 Comment=Insert shell command output into a document
 Comment[af]=Voeg by tolk opdrag uitset binnein 'n dokument
+Comment[ar]=أدخل مخرجات أمر من سطر الأوامر الى مستندك
 Comment[az]=Qabıq əmrə yekununu sənədə daxil et
 Comment[bg]=Вмъкване на резултата от изпълнението на дадена команда в документа
 Comment[bs]=Dodaj izlaz shell komande u dokument
@@ -62,7 +63,7 @@
 Comment[fr]=Pour insérer la sortie d'une commande shell dans un document
 Comment[gl]=Inseri-la saída dun comando de shell nun documento
 Comment[he]=הוספת פלט של פקודות מעטפת לתוך מסמך
-Comment[hi]=एक दस्तावेज में शैल कमांड आउटपुट प्रविष्ट करता है
+Comment[hi]=एक दस्तावेज़ में शैल कमांड आउटपुट प्रविष्ट करता है
 Comment[hr]=Umetni rezultat naredbe ljuske u dokument
 Comment[hu]=Parancs kimenetének beillesztése
 Comment[is]=Setja úttak skeljarskipunar inn í skjal
diff -urN -x CVS kdeaddons.orig/kate/kpybrowser/katepybrowse.desktop kdeaddons/kate/kpybrowser/katepybrowse.desktop
--- kdeaddons.orig/kate/kpybrowser/katepybrowse.desktop	Sun Feb 29 15:40:52 2004
+++ kdeaddons/kate/kpybrowser/katepybrowse.desktop	Sun Apr 25 00:51:05 2004
@@ -6,6 +6,7 @@
 X-Kate-Version=2.2
 Name=Kate Python Browser Plugin
 Name[af]=Kate Python Blaaier Inplak
+Name[ar]=ملحق متصفح Python لـKate
 Name[ca]=Connector del navegador Python per a Kate
 Name[cs]=Modul Python prohlížeče pro Kate
 Name[cy]=Ategyn porydd Python am Kate
@@ -30,7 +31,7 @@
 Name[nn]=Kate-programtillegg for Python-oversikt
 Name[pl]=Wtyczka przeglądarki do Pythona do Kate
 Name[pt]='Plugin' de Navegação no Python do Kate
-Name[pt_BR]=Plugin de Navegação Python do Kate
+Name[pt_BR]=Plug-in de Navegação Python do Kate
 Name[ro]=Modul Kate de navigare Python
 Name[ru]=Модуль просмотра программ на Python для Kate
 Name[sk]=Modul Kate Python prehliadača
@@ -48,6 +49,7 @@
 Name[zh_CN]=Kate Python 浏览器插件
 Comment=A Python browser plugin for Kate
 Comment[af]='n Python blaaier inplak vir Kate
+Comment[ar]=ملحق متصفح Python لـKate
 Comment[bg]=Приставка за разглеждане на класовете и методите на програми на Питон
 Comment[bs]=Python browser plugin za Kate
 Comment[ca]=Un connector per a Kate per a navegar dins del codi Python
@@ -75,7 +77,7 @@
 Comment[nn]=Ei Python-oversikt til Kate
 Comment[pl]=Wtyczka przeglądarki Pythona do Kate
 Comment[pt]=Um 'plugin' de navegação no Python para o Kate
-Comment[pt_BR]=Um Plugin de navegação Python para o Kate
+Comment[pt_BR]=Um Plug-in de navegação Python para o Kate
 Comment[ro]=Un modul de navigare Python pentru Kate
 Comment[ru]=Модуль просмотра программ на Python для Kate
 Comment[sk]=Python modul prehliadača pre Kate
diff -urN -x CVS kdeaddons.orig/kate/kpybrowser/ui.rc kdeaddons/kate/kpybrowser/ui.rc
--- kdeaddons.orig/kate/kpybrowser/ui.rc	Thu Dec 26 16:46:48 2002
+++ kdeaddons/kate/kpybrowser/ui.rc	Mon Apr 12 09:44:17 2004
@@ -1,7 +1,7 @@
 <!DOCTYPE kpartgui>
-<kpartplugin name="katepybrowse" library="libkatepybrowseplugin" version="2">
+<kpartplugin name="katepybrowse" library="libkatepybrowseplugin" version="3">
 <MenuBar>
-	<Menu name="python"><Text>&amp;Python</Text>
+	<Menu name="python"><Text>P&amp;ython</Text>
   	<Action name="python_update_pybrowse" />
   </Menu>
 </MenuBar>
diff -urN -x CVS kdeaddons.orig/kate/make/katemake.desktop kdeaddons/kate/make/katemake.desktop
--- kdeaddons.orig/kate/make/katemake.desktop	Fri Mar 12 13:41:07 2004
+++ kdeaddons/kate/make/katemake.desktop	Sun Apr 25 00:51:07 2004
@@ -4,6 +4,7 @@
 X-KDE-Library=katemakeplugin
 X-Kate-Version=2.2
 Name=Kate Make Plugin
+Name[ar]=ملحق Kate Make
 Name[ca]=Connector Make per a Kate
 Name[cs]=Make modul pro Kate
 Name[cy]=Ategyn Gwneud am Kate
@@ -25,7 +26,7 @@
 Name[nn]=Kate Make-programtillegg
 Name[pl]=Wtyczka Make dla Kate
 Name[pt]='Plugin' de Make do Kate
-Name[pt_BR]=Plugin de Make do Kate
+Name[pt_BR]=Plug-in de Make do Kate
 Name[ro]=Modul "make" pentru Kate
 Name[ru]=Модуль сборки для Kate
 Name[sk]=Kate Make modul
@@ -59,6 +60,7 @@
 Comment[it]=Compila e controlla i messaggi di errore
 Comment[ja]=エラー・メッセージをコンパイルして解析します
 Comment[ms]=Kompile dan mesej ralat huraian
+Comment[nb]=Kompiler og tolk feilmeldinger
 Comment[nl]=Compileer en ontleed foutmeldingen
 Comment[nn]=Kompiler og fortolk feilmeldingar
 Comment[pl]=Kompiluje i analizuje komunikaty o błędach
diff -urN -x CVS kdeaddons.orig/kate/modeline/katemodeline.desktop kdeaddons/kate/modeline/katemodeline.desktop
--- kdeaddons.orig/kate/modeline/katemodeline.desktop	Sun Feb 29 15:40:56 2004
+++ kdeaddons/kate/modeline/katemodeline.desktop	Sun Apr 25 00:51:09 2004
@@ -5,6 +5,7 @@
 X-Kate-Version=2.2
 Name=Kate Modeline Plugin
 Name[af]=Kate Moduslyn Inplak
+Name[ar]=ملحق Kate Modeline
 Name[ca]=Connector Modeline per a Kate
 Name[cy]=Ategyn Modd-linell Kate
 Name[da]=Kate Tilstandslinje-plugin
@@ -27,7 +28,7 @@
 Name[nn]=Kate nivålinje-tillegg
 Name[pl]=Wtyczka ustawień linii do Kate
 Name[pt]='Plugin' de Linha de Modo do Kate
-Name[pt_BR]=Plugin Modeline do Kate
+Name[pt_BR]=Plug-in Modeline do Kate
 Name[ro]=Modul "modeline" pentru Kate
 Name[ru]=Модуль строки состояния буфера для Kate
 Name[sk]=Kate Modeline modul
@@ -43,6 +44,7 @@
 Name[xx]=xxKate Modeline Pluginxx
 Name[zh_CN]=Kate 状态行插件
 Comment=Set document settings based on vim or emacs modelines
+Comment[ar]=عيّن اعدادات المستندات على أساس ملفات modelines الخاص بـvim أو emacs
 Comment[bg]=Установяване на режим базиран на редакторите vim или emacs
 Comment[bs]=Podešava postavke dokumenta na osnovu vim ili emacs modeline-ova
 Comment[ca]=Estableix les preferències dels documents basant-se en mode de línies de vim o emacs
@@ -57,7 +59,7 @@
 Comment[fo]=Nýt ásetingar fyri skjal grundað á vim ella emacs *modelines*
 Comment[fr]=Définit les réglages de document basés sur les modules de vim ou d'emacs
 Comment[he]=קביעת הגדרות המסמך בהתבסס על modelines של vim או emacs
-Comment[hi]=वीआईएम या ईमैक्स मॉडलाइन्स आधारित दस्तावेज विन्यास नियत करता है
+Comment[hi]=वीआईएम या ईमैक्स मॉडलाइन्स आधारित दस्तावेज़ विन्यास नियत करता है
 Comment[hr]=Podesi postavke dokumenta prema vim ili emacs modelines-ima
 Comment[hu]=Dokumentumbeállítások vim/emacs módleíró sorok alapján
 Comment[is]=Setur stillingar skjals byggt á vim eða emacs 'modeline'
diff -urN -x CVS kdeaddons.orig/kate/openheader/kateopenheader.desktop kdeaddons/kate/openheader/kateopenheader.desktop
--- kdeaddons.orig/kate/openheader/kateopenheader.desktop	Sun Feb 29 15:40:58 2004
+++ kdeaddons/kate/openheader/kateopenheader.desktop	Sun Apr 25 00:51:11 2004
@@ -36,6 +36,7 @@
 Name[zu]=u Kate Vula isihloko
 Comment=Opens the corresponding .h/[.cpp|.c] file
 Comment[af]=Maak oop die ooreenstemmend .h/[.cpp|.c] lêer
+Comment[ar]=تفتح ملف .h المقابل لملف [.cpp|.c]
 Comment[az]=Korespond .h/[.cpp|.c] faylını açar
 Comment[bg]=Отваряне на съответстващите файлове .h/[.cpp|.c]
 Comment[bs]=Otvara odgovarajuću .h/[.cpp|.c] datoteku
@@ -52,7 +53,7 @@
 Comment[fo]=Opnar tann samsvarandi .h/[.cpp|.c]-fíla.
 Comment[fr]=Pour ouvrir le fichier correspondant .h/[.cpp|.c]
 Comment[he]=פתיחת קובץ הכותרת המתאים [h/[.cpp|.c.
-Comment[hi]= .h/[.cpp|.c] के सदृश्य फाइल खोलता है
+Comment[hi]= .h/[.cpp|.c] के सदृश्य फ़ाइल खोलता है
 Comment[hr]=Otvori odgovarajuću .h/[.cpp|.c] datoteku
 Comment[hu]=A megfelelő .h/[.cpp|.c] fájl megnyitása
 Comment[is]=Opnar viðkomandi .h/[.cpp|.c] skrá
diff -urN -x CVS kdeaddons.orig/kate/scripts/html-tidy.desktop kdeaddons/kate/scripts/html-tidy.desktop
--- kdeaddons.orig/kate/scripts/html-tidy.desktop	Sun Feb 29 15:41:00 2004
+++ kdeaddons/kate/scripts/html-tidy.desktop	Sun Apr 25 00:51:13 2004
@@ -23,11 +23,12 @@
 Comment[fi]=Validoi nykyisen asiakirjan HTML Tidyllä
 Comment[fr]=Valide le document courant grâce à HTML Tidy
 Comment[gl]=Valida o documento actual mediante o HTML Tidy
-Comment[hi]=एचटीएमएल टाइडी के द्वारा वर्तमान दस्तावेज वेलिडेट करता है
+Comment[hi]=एचटीएमएल टाइडी के द्वारा वर्तमान दस्तावेज़ वेलिडेट करता है
 Comment[hu]=Az aktuális dokumentum ellenőrzése a HTML Tidy programmal
 Comment[is]=Staðfestir núverandi skjal með HTML Tidy
 Comment[it]=Convalida il documento attuale tramite HTML Tidy
 Comment[ja]=きちんとしたHTMLによって現在のドキュメントを有効にします
+Comment[nb]=Validerer gjeldende dokument via HTML Tidy
 Comment[nl]=Valideert het huidige document via HTML Tidy
 Comment[nn]=Validerer dokumentet med HTML Tidy
 Comment[pl]=Sprawdza poprawność kodu HTML aktualnego dokumentu za pomocą HTML Tidy
diff -urN -x CVS kdeaddons.orig/kate/textfilter/katetextfilter.desktop kdeaddons/kate/textfilter/katetextfilter.desktop
--- kdeaddons.orig/kate/textfilter/katetextfilter.desktop	Sun Feb 29 15:41:02 2004
+++ kdeaddons/kate/textfilter/katetextfilter.desktop	Sun Apr 25 00:51:15 2004
@@ -49,6 +49,7 @@
 Name[zu]=Ifayela Yokubhaliwe
 Comment=Easy text filtering
 Comment[af]=Maklike teks gefilter het
+Comment[ar]=نتقية سهلة للنص
 Comment[az]=Asan mətn süzülməsi
 Comment[bg]=Филтриране на текст чрез системата за вход-изход и пренасочване на команди в командния интерпретатор
 Comment[bs]=Lahko filtriranje teksta
diff -urN -x CVS kdeaddons.orig/kate/xmlcheck/katexmlcheck.desktop kdeaddons/kate/xmlcheck/katexmlcheck.desktop
--- kdeaddons.orig/kate/xmlcheck/katexmlcheck.desktop	Sun Feb 29 15:41:04 2004
+++ kdeaddons/kate/xmlcheck/katexmlcheck.desktop	Sun Apr 25 00:51:17 2004
@@ -45,6 +45,7 @@
 Name[xx]=xxKate XML Validationxx
 Name[zh_CN]=Kate XML 校验
 Comment=Validates XML files using xmllint
+Comment[ar]=يقوم بالتأكد من صلاحية ملفات XML باستخدام xmllint
 Comment[bg]=Валидиране на текущия документ XML чрез xmllint
 Comment[bs]=Provjerava ispravnost XML datoteka koristeći xmllint
 Comment[ca]=Valida fitxers XML emprant xmllint
@@ -59,7 +60,7 @@
 Comment[fr]=Valide un fichier XML en utilisant xmllint
 Comment[gl]=Valida os ficheiros XML empregando xmllinit
 Comment[he]=וידוא קבצי XML באמצעות xmllint
-Comment[hi]=xmllint के उपयोग से XML फाइलों को वेलिडेट करता है
+Comment[hi]=xmllint के उपयोग से XML फ़ाइलों को वेलिडेट करता है
 Comment[hu]=XML fájlok ellenőrzése az xmllint segítségével
 Comment[is]=Villuleitar XML skrár með xmllint
 Comment[it]=Valida i file XML usando xmllint
diff -urN -x CVS kdeaddons.orig/kfile-plugins/desktop/kfile_desktop.desktop kdeaddons/kfile-plugins/desktop/kfile_desktop.desktop
--- kdeaddons.orig/kfile-plugins/desktop/kfile_desktop.desktop	Sun Feb 29 15:41:08 2004
+++ kdeaddons/kfile-plugins/desktop/kfile_desktop.desktop	Sun Apr 25 00:51:19 2004
@@ -3,6 +3,7 @@
 Type=Service
 Name=Desktop Entry Info
 Name[af]=Werkskerm Inskrywing Inligting
+Name[ar]=معلومات مدخل سطح المكتب
 Name[bg]=Информация за файл Desktop
 Name[bs]=Informacije o desktop stavci
 Name[ca]=Informació de l'entrada a l'escriptori
diff -urN -x CVS kdeaddons.orig/kfile-plugins/folder/kfile_folder.desktop kdeaddons/kfile-plugins/folder/kfile_folder.desktop
--- kdeaddons.orig/kfile-plugins/folder/kfile_folder.desktop	Sun Feb 29 15:41:10 2004
+++ kdeaddons/kfile-plugins/folder/kfile_folder.desktop	Sun Apr 25 00:51:21 2004
@@ -3,6 +3,7 @@
 Type=Service
 Name=Folder Info
 Name[af]=Kabinet Inligting
+Name[ar]=معلومات المجلد
 Name[bg]=Информация за директория
 Name[bs]=Informacije o direktoriju
 Name[ca]=Info. de la carpeta
@@ -18,14 +19,14 @@
 Name[fr]=Informations sur le dossier
 Name[gl]=Información do Cartafol
 Name[he]=מידע תיקיה
-Name[hi]=फोल्डर जानकारी
+Name[hi]=फ़ोल्डर जानकारी
 Name[hu]=Mappajellemzők
 Name[is]=Möppuupplýsingar
 Name[it]=Informazioni su directory
 Name[ja]=フォルダ情報
 Name[lt]=Aplanko informacija
-Name[nb]=Kataloginformasjon
-Name[nl]=Mappen-informatie
+Name[nb]=Mappeinformasjon
+Name[nl]=Mappeninformatie
 Name[nn]=Kataloginformasjon
 Name[pl]=Informacja o katalogach
 Name[pt]=Informações de Pastas
diff -urN -x CVS kdeaddons.orig/kfile-plugins/html/kfile_html.desktop kdeaddons/kfile-plugins/html/kfile_html.desktop
--- kdeaddons.orig/kfile-plugins/html/kfile_html.desktop	Sun Feb 29 15:41:12 2004
+++ kdeaddons/kfile-plugins/html/kfile_html.desktop	Sun Apr 25 00:51:23 2004
@@ -2,6 +2,7 @@
 Type=Service
 Name=HTML Playlist Info
 Name[af]=Html Liedjielys Inligting
+Name[ar]=معلومات قائمة أغاني HTML
 Name[bg]=Информация за файл HTML
 Name[ca]=Info. de la llista de reproducció HTML
 Name[cs]=Informace o seznamu skladeb HTML
@@ -28,7 +29,7 @@
 Name[nn]=Informasjon om HTML-speleliste
 Name[pl]=Informacja o liście nagrań w HTML-u
 Name[pt]=Informações de Listas de Músicas HTML
-Name[pt_BR]=Informação de listas de reprodução HTML
+Name[pt_BR]=Informação de listas de músicas em HTML
 Name[ro]=Informaţii listă de redare HTML
 Name[ru]=Информация о списке песен в HTML
 Name[sk]=HTML informácie o playliste
diff -urN -x CVS kdeaddons.orig/kfile-plugins/txt/kfile_txt.desktop kdeaddons/kfile-plugins/txt/kfile_txt.desktop
--- kdeaddons.orig/kfile-plugins/txt/kfile_txt.desktop	Sun Feb 29 15:41:14 2004
+++ kdeaddons/kfile-plugins/txt/kfile_txt.desktop	Sun Apr 25 00:51:25 2004
@@ -3,6 +3,7 @@
 Type=Service
 Name=Text File Info
 Name[af]=Teks Lêer Inligting
+Name[ar]=معلومات ملف نصي
 Name[bg]=Информация за текстов файл
 Name[bs]=Info o tekst datoteci
 Name[ca]=Info. del fitxer de text
@@ -18,7 +19,7 @@
 Name[fo]=Tekstfíluupplýsingar
 Name[fr]=Informations sur le fichier texte
 Name[he]=מידע קובץ טקסט
-Name[hi]=पाठ फाइल जानकारी
+Name[hi]=पाठ फ़ाइल जानकारी
 Name[hr]=Informacije o tekst datoteci
 Name[hu]=Információ szöveges fájlokról
 Name[is]=Upplýsingar um textaskrá
@@ -30,7 +31,7 @@
 Name[nn]=Informasjon om tekstfil
 Name[pl]=Informacja o plikach tekstowych
 Name[pt]=Informações de Ficheiros de Texto
-Name[pt_BR]=Arquivo-texto
+Name[pt_BR]=Informações de Arquivo-texto
 Name[ro]=Informaţii fişier text
 Name[ru]=Информация о текстовом файле
 Name[sk]=Informácie o textovom súbore
diff -urN -x CVS kdeaddons.orig/kicker-applets/kbinaryclock/kbinaryclock.desktop kdeaddons/kicker-applets/kbinaryclock/kbinaryclock.desktop
--- kdeaddons.orig/kicker-applets/kbinaryclock/kbinaryclock.desktop	Fri Mar 12 13:41:11 2004
+++ kdeaddons/kicker-applets/kbinaryclock/kbinaryclock.desktop	Sun Apr 25 00:51:27 2004
@@ -1,5 +1,6 @@
 [Desktop Entry]
 Name=KBinaryClock
+Name[ar]=الساعة الثنائية
 Name[cs]=Binární hodiny
 Name[cy]=KClocDeuaidd
 Name[de]=Binäruhr
@@ -7,11 +8,12 @@
 Name[hi]=के-बाइनरी-क्लॉक
 Name[is]=Ktvíundarklukka
 Name[pt]=Relógio Binário
-Name[pt_BR]=Relógio Binário do KDE
+Name[pt_BR]=Relógio Binário
 Name[ro]=Ceas binar
 Name[sv]=Binärklocka
 Name[xx]=xxKBinaryClockxx
 Comment=Binary Clock
+Comment[ar]=ساعة ثنائية
 Comment[bg]=Двоичен часовник
 Comment[ca]=Rellotge binari
 Comment[cs]=Binární hodiny
@@ -31,6 +33,7 @@
 Comment[it]=Orologio binario
 Comment[ja]=kicker用のバイナリ時計
 Comment[ms]=Jam Binari
+Comment[nb]=Binær klokke
 Comment[nl]=Binaire klok
 Comment[nn]=Binærklokke
 Comment[pl]=Binarny zegar
diff -urN -x CVS kdeaddons.orig/kicker-applets/kolourpicker/kolourpicker.desktop kdeaddons/kicker-applets/kolourpicker/kolourpicker.desktop
--- kdeaddons.orig/kicker-applets/kolourpicker/kolourpicker.desktop	Sun Feb 29 15:41:18 2004
+++ kdeaddons/kicker-applets/kolourpicker/kolourpicker.desktop	Sun Apr 25 00:51:29 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=Color Picker
 Name[af]=Kleur Kieser
+Name[ar]=أداة اختيار الألوان
 Name[az]=Rəng Alıcısı
 Name[bg]=Избор на цвят
 Name[bs]=Izbornik boja
@@ -57,6 +58,7 @@
 Icon=colorpicker
 Comment=Pick colors from anywhere and get their values
 Comment[af]=Kies kleure van orals en kry hulle waardes
+Comment[ar]=اختر الألوان من أي مكان و احصل على قيمتها الرقمية
 Comment[az]=İstədiyiniz yerdən rəng alaraq qiymətlərini öyrənin
 Comment[bg]=Избор на цвят от екрана и получаване на неговата стойност
 Comment[bs]=Izbor boje sa bilo kojeg mjesta i ispis njenih vrijednosti
diff -urN -x CVS kdeaddons.orig/kicker-applets/ktimemon/ktimemon.desktop kdeaddons/kicker-applets/ktimemon/ktimemon.desktop
--- kdeaddons.orig/kicker-applets/ktimemon/ktimemon.desktop	Sun Feb 29 15:41:20 2004
+++ kdeaddons/kicker-applets/ktimemon/ktimemon.desktop	Sun Apr 25 00:51:31 2004
@@ -5,6 +5,7 @@
 X-KDE-UniqueApplet=true
 Comment=A system monitor for KDE
 Comment[af]='n stelsel monitor vir Kde
+Comment[ar]=أداة مراقبة النظام لكيدي
 Comment[az]=KDE üçün sistem izləyici vasitə
 Comment[bg]=Системен монитор за КДЕ
 Comment[bs]=Stanje sistema za KDE
@@ -61,6 +62,7 @@
 Comment[zu]=Umphendli wendlela esetshenziswayo ye KDE
 Name=System Monitor
 Name[af]=Stelsel Monitor
+Name[ar]=مراقب النظام
 Name[az]=Sistem İzləyici
 Name[bg]=Системен монитор
 Name[bs]=Stanje sistema
diff -urN -x CVS kdeaddons.orig/kicker-applets/mediacontrol/mediacontrol.desktop kdeaddons/kicker-applets/mediacontrol/mediacontrol.desktop
--- kdeaddons.orig/kicker-applets/mediacontrol/mediacontrol.desktop	Fri Mar 12 13:41:13 2004
+++ kdeaddons/kicker-applets/mediacontrol/mediacontrol.desktop	Sun Apr 25 00:51:33 2004
@@ -40,6 +40,7 @@
 Name[zu]=Ulawulo Lwezindaba
 Comment=Applet to control mediaplayers
 Comment[af]=Miniprogram na kontrole media spelers
+Comment[ar]=بريمج للتحكم بمشغلات الوسائط المتعددة
 Comment[az]=Konsol medya çalğıcıları appleti
 Comment[bg]=Лесно и универсално управление на плеъри за мултимедийни файлове
 Comment[bs]=Applet za kontrolu mediaplayera
diff -urN -x CVS kdeaddons.orig/konq-plugins/babelfish/babelfishplugin.desktop kdeaddons/konq-plugins/babelfish/babelfishplugin.desktop
--- kdeaddons.orig/konq-plugins/babelfish/babelfishplugin.desktop	Sun Feb 29 15:41:25 2004
+++ kdeaddons/konq-plugins/babelfish/babelfishplugin.desktop	Sun Apr 25 00:51:35 2004
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Babel plugin
+Name[ar]=ملحق Babel
 Name[bg]=Приставка за Babel
 Name[ca]=Connector per a Babel
 Name[cs]=Modul Babel
@@ -41,6 +42,7 @@
 Name[xx]=xxBabel pluginxx
 Name[zh_CN]=Babel 插件
 Comment=Babelfish plugin
+Comment[ar]=ملحق Babelfish
 Comment[bg]=Приставка за програмата за превеждане Babelfish
 Comment[ca]=Connector per a Babelfish
 Comment[cs]=Modul Babelfish
diff -urN -x CVS kdeaddons.orig/konq-plugins/crashes/crashesplugin.desktop kdeaddons/konq-plugins/crashes/crashesplugin.desktop
--- kdeaddons.orig/konq-plugins/crashes/crashesplugin.desktop	Sun Feb 29 15:41:27 2004
+++ kdeaddons/konq-plugins/crashes/crashesplugin.desktop	Sun Apr 25 00:51:37 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=Crashes monitor
 Name[af]=Verongeluk monitor
+Name[ar]=مراقب الإنهيارات
 Name[bg]=Монитор на блокирали процеси
 Name[ca]=Penja el monitor
 Name[cs]=Monitor pádů
@@ -25,7 +26,7 @@
 Name[nn]=Krasjovervaking
 Name[pl]=Monitor awarii programu
 Name[pt]=Monitor de estoiros
-Name[pt_BR]=Monitor de problemas
+Name[pt_BR]=Monitor de quedas de programas
 Name[ro]=Monitor de prăbuşiri
 Name[ru]=Монитор сбоев
 Name[sk]=Monitor pádov
@@ -41,6 +42,7 @@
 Name[zh_CN]=崩溃监视器
 Comment=Crashes monitor
 Comment[af]=Verongeluk monitor
+Comment[ar]=مراقب الإنهيارات
 Comment[bg]=Монитор на блокирали процеси за КДЕ
 Comment[ca]=Penja el monitor
 Comment[cs]=Monitor pádů
@@ -66,7 +68,7 @@
 Comment[nn]=Overvakar krasj
 Comment[pl]=Monitor awarii programu
 Comment[pt]=Monitor de estoiros
-Comment[pt_BR]=Monitor de sistema
+Comment[pt_BR]=Monitor de problemas no sistema
 Comment[ro]=Program de monitorizare a prăbuşirii programelor
 Comment[ru]=Монитор сбоев
 Comment[sk]=Monitor pádov
diff -urN -x CVS kdeaddons.orig/konq-plugins/dirfilter/dirfilterplugin.desktop kdeaddons/konq-plugins/dirfilter/dirfilterplugin.desktop
--- kdeaddons.orig/konq-plugins/dirfilter/dirfilterplugin.desktop	Sun Feb 29 15:41:29 2004
+++ kdeaddons/konq-plugins/dirfilter/dirfilterplugin.desktop	Sun Apr 25 00:51:39 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=Directory filter plugin
 Name[af]=Gids filter inplak
+Name[ar]=ملحق تنقية المجلدات
 Name[bg]=Приставка за филтриране на директории
 Name[bs]=Plugin za filtriranje direktorija
 Name[ca]=Connector filtre del directori
@@ -21,12 +22,12 @@
 Name[it]=Plugin per il filtro delle directory
 Name[ja]=ディレクトリフィルタプラグイン
 Name[lt]=Aplanko filtro priedas
-Name[nb]=Progamtillegg til katalogfilter
+Name[nb]=Progamtillegg til mappefilter
 Name[nl]=Mappenfilterplugin
 Name[nn]=Progamtillegg for katalogfilter
 Name[pl]=Wtyczka filtrowania katalogów
 Name[pt]='Plugin' de filtragem de directorias
-Name[pt_BR]=Diretório de plugin de filtro
+Name[pt_BR]=Plug-in de filtro de pastas
 Name[ro]=Modul filtrare directoare
 Name[ru]=Фильтр каталогов
 Name[sk]=Modul adresárového filtra
@@ -43,6 +44,7 @@
 Name[zh_CN]=目录过滤器插件
 Comment=Directory filter plugin
 Comment[af]=Gids filter inplak
+Comment[ar]=ملحق تنقية المجلدات
 Comment[bg]=Приставка за филтриране на директории
 Comment[bs]=Plugin za filtriranje direktorija
 Comment[ca]=Connector filtre del directori
@@ -62,12 +64,12 @@
 Comment[it]=Plugin per il filtro delle directory
 Comment[ja]=ディレクトリフィルタプラグイン
 Comment[lt]=Aplanko filtro priedas
-Comment[nb]=Progamtillegg til katalogfilter
+Comment[nb]=Progamtillegg til mappefilter
 Comment[nl]=Mappenfilterplugin
 Comment[nn]=Progamtillegg for katalogfilter
 Comment[pl]=Wtyczka filtrowania katalogów
 Comment[pt]='Plugin' de filtragem de directorias
-Comment[pt_BR]=Diretório de Plugin de filtro
+Comment[pt_BR]=Plug-in de filtro
 Comment[ro]=Modul de filtrare a directoarelor de afişat
 Comment[ru]=Модуль фильтра каталогов
 Comment[sk]=Modul adresárového filtra
diff -urN -x CVS kdeaddons.orig/konq-plugins/domtreeviewer/domtreeviewerplugin.desktop kdeaddons/konq-plugins/domtreeviewer/domtreeviewerplugin.desktop
--- kdeaddons.orig/konq-plugins/domtreeviewer/domtreeviewerplugin.desktop	Sun Feb 29 15:41:31 2004
+++ kdeaddons/konq-plugins/domtreeviewer/domtreeviewerplugin.desktop	Sun Apr 25 00:51:41 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=DOM tree viewer
 Name[af]=Dom boom aansig
+Name[ar]=مستعرض شجرةDOM
 Name[bg]=Преглед на DOM
 Name[bs]=Preglednik DOM stabla
 Name[ca]=Visor de l'arbre DOM
@@ -43,6 +44,7 @@
 Name[zh_CN]=DOM 树形视图查看器
 Comment=DOM tree viewer
 Comment[af]=Dom boom aansig
+Comment[ar]=مستعرض شجرة DOM
 Comment[bg]=Преглед на структурата на DOM
 Comment[bs]=Preglednik DOM stabla
 Comment[ca]=Visor de l'arbre DOM
diff -urN -x CVS kdeaddons.orig/konq-plugins/fsview/fsview.desktop kdeaddons/konq-plugins/fsview/fsview.desktop
--- kdeaddons.orig/konq-plugins/fsview/fsview.desktop	Fri Mar 12 13:41:15 2004
+++ kdeaddons/konq-plugins/fsview/fsview.desktop	Sun Apr 25 00:51:43 2004
@@ -9,7 +9,6 @@
 Terminal=0
 Name=FSView
 Name[hi]=एफएस-व्यू
-Name[pt_BR]=Visão do SistArq
 Name[ro]=Vizualizor sistem de fişiere
 Name[sv]=Fsview
 Name[ta]=Kகாட்சி
@@ -27,11 +26,12 @@
 Comment[fr]=Afficher votre système de fichiers sous forme d'une arborescence
 Comment[gl]=Vexa o seu sistema de ficheiros en forma de Árbores
 Comment[he]=תצוגת הקבצים שלך בתור עץ
-Comment[hi]=आपके फाइल-सिस्टम को एक ट्री-मैप में दिखाए
+Comment[hi]=आपके फ़ाइल-सिस्टम को एक ट्री-मैप में दिखाए
 Comment[hu]=A fájlrendszer áttekintése fastruktúraként
 Comment[is]=Flakka um skráarkerfið í tráarham
 Comment[it]=Visualizza il tuo filesystem come una mappa ad albero
 Comment[ja]=ツリーマップでファイルシステムを表示
+Comment[nb]=Se filsystemet som et tre-kart
 Comment[nl]=Bekijk uw bestandssysteem als boomstructuur
 Comment[nn]=Viser filsystemet som eit trekart
 Comment[pl]=Przedstawia system plików jako drzewo
diff -urN -x CVS kdeaddons.orig/konq-plugins/fsview/fsview_part.desktop kdeaddons/konq-plugins/fsview/fsview_part.desktop
--- kdeaddons.orig/konq-plugins/fsview/fsview_part.desktop	Sun Jan 11 08:27:38 2004
+++ kdeaddons/konq-plugins/fsview/fsview_part.desktop	Sun Apr 18 14:31:49 2004
@@ -5,7 +5,6 @@
 Name[et]= FSView komponent
 Name[hi]=एफएस-व्यू-पार्ट
 Name[hu]=térképszerű nézet
-Name[pt_BR]=Componente de visualização do SistArq
 Name[ro]=Modul vizualizor sistem de fişiere
 Name[sv]=Fsview-del
 Name[ta]=FSகாட்சிபகுதி
diff -urN -x CVS kdeaddons.orig/konq-plugins/imagerotation/jpegorient.desktop kdeaddons/konq-plugins/imagerotation/jpegorient.desktop
--- kdeaddons.orig/konq-plugins/imagerotation/jpegorient.desktop	Fri Mar 12 13:41:17 2004
+++ kdeaddons/konq-plugins/imagerotation/jpegorient.desktop	Sun Apr 18 14:31:51 2004
@@ -23,6 +23,7 @@
 Name[is]=Snúa réttsælis
 Name[it]=Ruota in senso orario
 Name[ja]=右回りに回転します
+Name[nb]=Roter medurs
 Name[nl]=Klokgewijs draaien
 Name[nn]=Roter med klokka
 Name[pl]=Obróć zgodnie z ruchem wskazówek zegara
@@ -64,11 +65,12 @@
 Name[is]=Snúa rangsælis
 Name[it]=Ruota in senso anti-orario
 Name[ja]=左回りに回転します
+Name[nb]=Roter moturs
 Name[nl]=Anti-klokgewijs draaien
 Name[nn]=Roter mot klokka
 Name[pl]=Obróć przeciwnie do ruchu wskazówek zegara
 Name[pt]=Rodar no Sentido Antihorário
-Name[pt_BR]=Rodar Contador de Relógio
+Name[pt_BR]=Rodar Relógio Sentido Horário
 Name[ro]=Roteşte la stînga
 Name[ru]=Вращать против часовой стрелки
 Name[sk]=Otočiť proti smeru hodinových ručičiek
@@ -110,6 +112,7 @@
 Name[is]=Snúa við um miðju lóðrétt
 Name[it]=Fletti verticalmente
 Name[ja]=縦のフリップ
+Name[nb]=Speilvend loddrett
 Name[nl]=Verticaal omdraaien
 Name[nn]=Snu loddrett
 Name[pl]=Odwróć pionowo
@@ -151,6 +154,7 @@
 Name[is]=Snúa við um miðju lárétt
 Name[it]=Fletti orizzontalmente
 Name[ja]=水平にフリップ
+Name[nb]=Speilvend vannrett
 Name[nl]=Horizontaal omdraaien
 Name[nn]=Snu vassrett
 Name[pl]=Odwróć poziomo
diff -urN -x CVS kdeaddons.orig/konq-plugins/khtmlsettingsplugin/khtmlsettingsplugin.desktop kdeaddons/konq-plugins/khtmlsettingsplugin/khtmlsettingsplugin.desktop
--- kdeaddons.orig/konq-plugins/khtmlsettingsplugin/khtmlsettingsplugin.desktop	Sun Mar  7 14:17:55 2004
+++ kdeaddons/konq-plugins/khtmlsettingsplugin/khtmlsettingsplugin.desktop	Sun Apr 25 00:51:45 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=KHTML Settings
 Name[af]=Khtml Instellings
+Name[ar]=اعدادات KHTML
 Name[bs]=KHTML postavke
 Name[ca]=Preferències KHTML
 Name[cs]=Nastavení KHTML
@@ -44,6 +45,7 @@
 Name[zh_CN]=KHTML 设置
 Comment=Fast way to change the KHTML settings
 Comment[af]=Vinnige weg na verander die Khtml instellings
+Comment[ar]=طريقة سهلة لتغيير اعدادات KHTML
 Comment[bg]=Бърз начин да се променят някои от най-важните настройки на модула KHTML
 Comment[bs]=Brz način za mijenjanje KHTML postavki
 Comment[ca]=Per a canviar ràpidament les preferències de KHTML
diff -urN -x CVS kdeaddons.orig/konq-plugins/kimgalleryplugin/imgalleryplugin.cpp kdeaddons/konq-plugins/kimgalleryplugin/imgalleryplugin.cpp
--- kdeaddons.orig/konq-plugins/kimgalleryplugin/imgalleryplugin.cpp	Wed Nov  5 13:08:39 2003
+++ kdeaddons/konq-plugins/kimgalleryplugin/imgalleryplugin.cpp	Wed Apr 14 13:02:21 2004
@@ -39,6 +39,7 @@
 #include <kdebug.h>
 #include <kgenericfactory.h>
 #include <kio/netaccess.h>
+#include <kpushbutton.h>
 
 #include "imgallerydialog.h"
 #include "imgalleryplugin.h"
@@ -82,7 +83,7 @@
             QObject::connect(m_progressDlg, SIGNAL( cancelled() ), this, SLOT( slotCancelled() ) );
 
             m_progressDlg->setLabelText( i18n("Creating thumbnails") );
-            m_progressDlg->setCancelButtonText(i18n("&Cancel"));
+            m_progressDlg->setCancelButton(new KPushButton(KStdGuiItem::cancel(),m_progressDlg));
             m_cancelled = false;
             m_progressDlg->show();
             if ( createHtml( url, m_part->url().path(), m_configDlg->recursionLevel() > 0 ? m_configDlg->recursionLevel() + 1 : 0 , m_configDlg->getImageFormat()) ) {
diff -urN -x CVS kdeaddons.orig/konq-plugins/kimgalleryplugin/kimgalleryplugin.desktop kdeaddons/konq-plugins/kimgalleryplugin/kimgalleryplugin.desktop
--- kdeaddons.orig/konq-plugins/kimgalleryplugin/kimgalleryplugin.desktop	Sun Feb 29 15:41:39 2004
+++ kdeaddons/konq-plugins/kimgalleryplugin/kimgalleryplugin.desktop	Sun Apr 25 00:51:47 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=Image gallery
 Name[af]=Beeld galery
+Name[ar]=معرض الصور
 Name[bs]=Galerija slika
 Name[ca]=Galeria d'imatges
 Name[cs]=Obrázková galerie
@@ -16,7 +17,7 @@
 Name[fr]=Galerie d'images
 Name[gl]=Galería de Imaxes
 Name[he]=גלריית תמונות
-Name[hi]=चित्र दीर्घा
+Name[hi]=छवि दीर्घा
 Name[hu]=Képbemutató
 Name[is]=Myndasafn
 Name[it]=Galleria di immagini
@@ -44,6 +45,7 @@
 Name[zh_CN]=图像集
 Comment=Easy way to generate a HTML image gallery
 Comment[af]=Maklike weg na genereer 'n Html beeld galery
+Comment[ar]=طريقة سهلة لانشاء معرض صور HTML
 Comment[bg]=Лесен начин за генериране на галерия от изображения във формат HTML
 Comment[bs]=Lagan način za generisanje HTML galerija slika
 Comment[ca]=Genera fàcilment una galeria HTML d'imatges
@@ -58,7 +60,7 @@
 Comment[fr]=Manière rapide pour configurer une galerie d'images HTML
 Comment[gl]=Xeito doado de xerar unha galería de imaxes en HTML
 Comment[he]=דרך קלה ליצור גלריית תמונות מבוססת HTML
-Comment[hi]=एचटीएमएल चित्र दीर्घा बनाने का तेज तरीका
+Comment[hi]=एचटीएमएल छवि दीर्घा बनाने का तेज तरीका
 Comment[hu]=HTML-alapú képmegjelenítő
 Comment[is]=Einföld leið til að búa til HTML myndasöfn
 Comment[it]=Modo facile per generare una galleria di immagini in HTML
diff -urN -x CVS kdeaddons.orig/konq-plugins/kuick/kcmkuick/kcmkuick.desktop kdeaddons/konq-plugins/kuick/kcmkuick/kcmkuick.desktop
--- kdeaddons.orig/konq-plugins/kuick/kcmkuick/kcmkuick.desktop	Sun Mar  7 14:17:57 2004
+++ kdeaddons/konq-plugins/kuick/kcmkuick/kcmkuick.desktop	Sun Apr 25 00:51:49 2004
@@ -7,6 +7,7 @@
 X-KDE-Library=kuick
 Name=Quick Copy & Move Plugin
 Name[af]=Vinnige Kopie & Beweeg Inplak
+Name[ar]=ملحق النقل و النسخ السريع
 Name[az]=Əlavəni Sürətli Köçür və Daşı
 Name[bs]=Dodatak za brzo kopiranje i premještanje
 Name[ca]=Connector còpia i mou ràpidament
@@ -36,7 +37,7 @@
 Name[nso]=Plugin ya Kapela ya Gatisa & Sutisa
 Name[pl]=Szybkie kopiowanie i przesuwanie
 Name[pt]='Plugin' de Cópia e Mudança Rápida
-Name[pt_BR]= Plugin de Copiar & Mover Rápido
+Name[pt_BR]= Plug-in de Copiar & Mover Rápido
 Name[ro]=Modul de copiere şi mutare rapidă
 Name[ru]=Быстрый перенос и копирование файлов
 Name[sk]=Modul pre rýchle kopírovanie a presun
@@ -57,6 +58,7 @@
 Name[zu]=Kuick Khiphela futhi Ususe i Plugin
 Comment=Kuick configuration module
 Comment[af]=Vinnige opstelling module
+Comment[ar]=أداة تهيئة Kuick
 Comment[az]=Kuick quraşdırma modulu
 Comment[bg]=Конфигурационен модул за Kuick
 Comment[bs]=Modul za podešavanje Kuick-a
@@ -74,7 +76,7 @@
 Comment[ga]=Modúl cumraíochta Kuick
 Comment[gl]=Módulo de configuración rápida
 Comment[he]=שינוי הגדרות התוסף Kuick
-Comment[hi]=क्विक कॉनफिगरेशन मॉड्यूल
+Comment[hi]=क्विक कॉन्फ़िगरेशन मॉड्यूल
 Comment[hr]=Alat za podešavanje Kuick modula
 Comment[hu]=A Kuick beállítómodulja
 Comment[is]=Stillingatól Kuick
diff -urN -x CVS kdeaddons.orig/konq-plugins/kuick/kmetamenu.cpp kdeaddons/konq-plugins/kuick/kmetamenu.cpp
--- kdeaddons.orig/konq-plugins/kuick/kmetamenu.cpp	Tue Oct 21 23:31:23 2003
+++ kdeaddons/konq-plugins/kuick/kmetamenu.cpp	Thu Apr 15 14:23:15 2004
@@ -134,7 +134,7 @@
      conf->sync( );
 }
 void KMetaMenu::slotBrowse() {
-     QString dest = KFileDialog::getExistingDirectory();
+     KURL dest = KFileDialog::getExistingURL();
      if( dest.isEmpty() ) return;
-     slotFileChosen( dest );
+     slotFileChosen( dest.isLocalFile() ? dest.path() : dest.url() );
 }
diff -urN -x CVS kdeaddons.orig/konq-plugins/kuick/kuick_plugin.cpp kdeaddons/konq-plugins/kuick/kuick_plugin.cpp
--- kdeaddons.orig/konq-plugins/kuick/kuick_plugin.cpp	Sat Jul 26 22:04:47 2003
+++ kdeaddons/konq-plugins/kuick/kuick_plugin.cpp	Thu Apr 15 14:23:15 2004
@@ -57,13 +57,13 @@
 
 }
 void KTestMenu::slotStartCopyJob( const QString &path ) {
-     KURL url( path );
+     KURL url = KURL::fromPathOrURL( path );
      KIO::CopyJob *copy;
      copy = KIO::copy( popup->popupURLList(), url);
      copy->setAutoErrorHandlingEnabled( true );
 }
 void KTestMenu::slotStartMoveJob( const QString &path) {
-     KURL url( path );
+     KURL url = KURL::fromPathOrURL( path );
      KIO::CopyJob *move;
      move = KIO::move( popup->popupURLList(), url );
      move->setAutoErrorHandlingEnabled( true );
diff -urN -x CVS kdeaddons.orig/konq-plugins/kuick/kuick_plugin.desktop kdeaddons/konq-plugins/kuick/kuick_plugin.desktop
--- kdeaddons.orig/konq-plugins/kuick/kuick_plugin.desktop	Sun Feb 29 15:41:43 2004
+++ kdeaddons/konq-plugins/kuick/kuick_plugin.desktop	Sun Apr 18 14:31:55 2004
@@ -30,7 +30,7 @@
 Name[nso]=Tshutiso le Kgatiso ya Kuick
 Name[pl]=Szybkie kopiowanie i przesuwanie
 Name[pt]=Cópia e Mudança Rápida
-Name[pt_BR]=Lançador rápido
+Name[pt_BR]=Copiar e Mover
 Name[ro]=Mutare şi copiere Kuick
 Name[ru]=Быстрое копирование - Kuick
 Name[sk]=Kuick Kopírovanie a Presun
diff -urN -x CVS kdeaddons.orig/konq-plugins/kuick/kuickplugin.desktop kdeaddons/konq-plugins/kuick/kuickplugin.desktop
--- kdeaddons.orig/konq-plugins/kuick/kuickplugin.desktop	Sun Feb 29 15:41:45 2004
+++ kdeaddons/konq-plugins/kuick/kuickplugin.desktop	Sun Apr 18 14:31:57 2004
@@ -29,7 +29,7 @@
 Name[nso]=Tshutiso le Kgatiso ya Kuick
 Name[pl]=Szybkie kopiowanie i przesuwanie
 Name[pt]=Cópia e Mudança Rápida
-Name[pt_BR]=Lançador rápido
+Name[pt_BR]=Copiar e Mover
 Name[ro]=Mutare şi copiere Kuick
 Name[ru]=Быстрое копирование - Kuick
 Name[sk]=Kuick Kopírovanie a Presun
diff -urN -x CVS kdeaddons.orig/konq-plugins/minitools/minitoolsplugin.desktop kdeaddons/konq-plugins/minitools/minitoolsplugin.desktop
--- kdeaddons.orig/konq-plugins/minitools/minitoolsplugin.desktop	Sun Mar  7 14:17:59 2004
+++ kdeaddons/konq-plugins/minitools/minitoolsplugin.desktop	Sun Apr  4 14:53:32 2004
@@ -16,6 +16,7 @@
 Name[is]=Lítil tól
 Name[it]=Strumenti miniaturizzati
 Name[ja]=ミニツール
+Name[nb]=Mini-verktøy
 Name[nn]=Miniverktøy
 Name[pl]=Mini-narzędzia
 Name[pt]=Mini-Ferramentas
@@ -49,6 +50,7 @@
 Comment[is]=Lítil tól
 Comment[it]=Strumenti miniaturizzati
 Comment[ja]=ミニツール
+Comment[nb]=Mini-verktøy
 Comment[nl]=Mini-hulpmiddelen
 Comment[nn]=Miniverktøy
 Comment[pl]=Mini-narzędzia
diff -urN -x CVS kdeaddons.orig/konq-plugins/sidebar/mediaplayer/mediaplayerplugin.desktop kdeaddons/konq-plugins/sidebar/mediaplayer/mediaplayerplugin.desktop
--- kdeaddons.orig/konq-plugins/sidebar/mediaplayer/mediaplayerplugin.desktop	Sun Feb 29 15:41:49 2004
+++ kdeaddons/konq-plugins/sidebar/mediaplayer/mediaplayerplugin.desktop	Sun Apr 25 00:51:51 2004
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Mediaplayer
+Name[ar]=مشغل الوسائط المتعددة
 Name[ca]=Reproductor Multimèdia
 Name[cs]=Přehrávač médií
 Name[cy]=Chwaraewr Cyfryngau
@@ -21,7 +22,6 @@
 Name[nn]=Mediespelar
 Name[nso]=Sebapadi sa Media
 Name[pl]=Odtwarzacz multimedialny
-Name[pt_BR]=Reprodutor de Mídia
 Name[ro]=Redare multimedia
 Name[ru]=Медиапроигрыватель
 Name[sk]=Prehrávač médií
@@ -33,6 +33,7 @@
 Name[xx]=xxMediaplayerxx
 Name[zh_CN]=媒体播放器
 Comment=Mediaplayer plugin in Konqueror's navigation panel
+Comment[ar]=ملحق يشغل الوسائط المتعددة في لوحة التنقل في كونكيورر
 Comment[bg]=Навигационен панел за плеър за мултимедийни файлове
 Comment[bs]=Mediaplayer dodatak za Konqueror-ov navigatorski panel
 Comment[ca]=Connector del reproductor multimèdia en el plafó de navegació de Konqueror
@@ -58,7 +59,7 @@
 Comment[nn]=Mediespelar-programtillegg til navigasjonspanelet i Konqueror
 Comment[pl]=Wtyczka odtwarzacza multimediów w panelu nawigacyjnym Konquerora
 Comment[pt]=Um 'plugin' de reprodução multimédia no painel de navegação do Konqueror
-Comment[pt_BR]=plug-ins do reprodutor de mídia no painel de navegação do Konqueror
+Comment[pt_BR]=Plug-in do reprodutor de mídia no painel de navegação do Konqueror
 Comment[ro]=Modul de redare multimedia în panoul de navigare Konqueror
 Comment[ru]=Модуль Konqueror для проигрывания мультимедиа
 Comment[sk]=Mediaplayer modul v navigačnom panely Konquerora
diff -urN -x CVS kdeaddons.orig/konq-plugins/sidebar/mediaplayer/mplayer.desktop kdeaddons/konq-plugins/sidebar/mediaplayer/mplayer.desktop
--- kdeaddons.orig/konq-plugins/sidebar/mediaplayer/mplayer.desktop	Sun Feb 29 15:41:51 2004
+++ kdeaddons/konq-plugins/sidebar/mediaplayer/mplayer.desktop	Sun Apr 25 00:51:53 2004
@@ -5,6 +5,7 @@
 Icon=konqsidebar_mediaplayer
 Name=Sidebar Media Player
 Name[af]=Kantbalk Media Speler
+Name[ar]=مشغل الوسائط المتعددة للشريط الجانبي
 Name[az]=Yan Çubuq Medya Çalğıcısı
 Name[ca]=Reproductor multimèdia per al plafó de navegació
 Name[cs]=Přehrávač médií pro boční panel
@@ -36,7 +37,7 @@
 Name[nso]=Sebapadi sa Media wa Bar ya Lehlakori
 Name[pl]=Odtwarzacz mediów w pasku bocznym
 Name[pt]=Reprodutor Multimédia da Barra Lateral
-Name[pt_BR]=Barra Lateral do Tocador de Mídia
+Name[pt_BR]=Barra Lateral do Media Player
 Name[ro]=Bară laterală redare multimedia
 Name[ru]=Медиаплеер боковой панели
 Name[sk]=Prehrávač médií pre bočný panel
diff -urN -x CVS kdeaddons.orig/konq-plugins/sidebar/mediaplayer/mplayer_add.desktop kdeaddons/konq-plugins/sidebar/mediaplayer/mplayer_add.desktop
--- kdeaddons.orig/konq-plugins/sidebar/mediaplayer/mplayer_add.desktop	Sun Feb 29 15:41:53 2004
+++ kdeaddons/konq-plugins/sidebar/mediaplayer/mplayer_add.desktop	Sun Apr 25 00:51:55 2004
@@ -5,6 +5,7 @@
 Icon=konqsidebar_mediaplayer
 Name=Sidebar Media Player
 Name[af]=Kantbalk Media Speler
+Name[ar]=مشغل الوسائط المتعددة للشريط الجانبي
 Name[az]=Yan Çubuq Medya Çalğıcısı
 Name[ca]=Reproductor multimèdia per al plafó de navegació
 Name[cs]=Přehrávač médií pro boční panel
@@ -36,7 +37,7 @@
 Name[nso]=Sebapadi sa Media wa Bar ya Lehlakori
 Name[pl]=Odtwarzacz mediów w pasku bocznym
 Name[pt]=Reprodutor Multimédia da Barra Lateral
-Name[pt_BR]=Barra Lateral do Tocador de Mídia
+Name[pt_BR]=Barra Lateral do Media Player
 Name[ro]=Bară laterală redare multimedia
 Name[ru]=Медиаплеер боковой панели
 Name[sk]=Prehrávač médií pre bočný panel
diff -urN -x CVS kdeaddons.orig/konq-plugins/smbmounter/smbmounterplugin.desktop kdeaddons/konq-plugins/smbmounter/smbmounterplugin.desktop
--- kdeaddons.orig/konq-plugins/smbmounter/smbmounterplugin.desktop	Fri Mar 12 13:41:19 2004
+++ kdeaddons/konq-plugins/smbmounter/smbmounterplugin.desktop	Sun Apr 25 00:51:57 2004
@@ -18,6 +18,7 @@
 Name[is]=SMB tengjari
 Name[ja]=SMBマウンタ
 Name[ms]=Pelekap SMB
+Name[nb]=SMB-monterer
 Name[nn]=SMB-montering
 Name[pl]=Program montujący udziały SMB
 Name[pt]=Montar SMB
@@ -34,6 +35,7 @@
 Name[xx]=xxSMB Mounterxx
 Name[zh_CN]=SMB 挂载器
 Comment=SMB mounter plugin
+Comment[ar]=ملحق SMB mounter
 Comment[bg]=Приставка за монтиране на споделени ресурси SMB
 Comment[ca]=Connector per a muntar SMB
 Comment[cs]=Modul pro připojování SMB
@@ -59,7 +61,7 @@
 Comment[nn]=Programtillegg for SMB-montering
 Comment[pl]=Wtyczka montowania udziałów SMB
 Comment[pt]='Plugin' de montagem de SMB
-Comment[pt_BR]=Plugin de montagem de partições SMB
+Comment[pt_BR]=Plug-in de montagem de partições SMB
 Comment[ro]=Modul de montare partajări SMB (Samba)
 Comment[ru]=Модуль подключения ресурсов Samba
 Comment[sk]=Modul na pripájanie SMB súborového systému
diff -urN -x CVS kdeaddons.orig/konq-plugins/uachanger/uachangerplugin.desktop kdeaddons/konq-plugins/uachanger/uachangerplugin.desktop
--- kdeaddons.orig/konq-plugins/uachanger/uachangerplugin.desktop	Sun Feb 29 15:41:57 2004
+++ kdeaddons/konq-plugins/uachanger/uachangerplugin.desktop	Sun Apr 25 00:51:59 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=UserAgent changer
 Name[af]=Gebruiker agent wisselaar
+Name[ar]=أداة تغيير عميل المستخدم
 Name[ca]=Canvia l'agent d'usuari
 Name[cs]=Změna uživatelského agenta
 Name[cy]=Newidydd Asiant y Defnyddiwr
@@ -25,7 +26,7 @@
 Name[nn]=Endring av brukaragent
 Name[pl]=Zmiana identyfikacji przeglądarki
 Name[pt]=Mudar agente do utilizador
-Name[pt_BR]=Modificar de Agente de usuários
+Name[pt_BR]=Modificador de Agente de usuários
 Name[ro]=Modificare "User Agent"
 Name[ru]=Псевдонимы браузера
 Name[sk]=Zmena UserAgent-a
@@ -41,6 +42,7 @@
 Name[zh_CN]=用户代理更换器
 Comment=Plugin for changing the user agent
 Comment[af]=Inplak vir verandering die gebruiker agent
+Comment[ar]=ملحق لتغيير عميل المستخدم
 Comment[bg]=Приставка за смяна на идентификацията на браузъра
 Comment[bs]=Dodatak za promijenju user agent-a
 Comment[ca]=Connector per a modificar l'agent d'usuari
diff -urN -x CVS kdeaddons.orig/konq-plugins/validators/validatorsplugin.desktop kdeaddons/konq-plugins/validators/validatorsplugin.desktop
--- kdeaddons.orig/konq-plugins/validators/validatorsplugin.desktop	Sun Feb 29 15:41:59 2004
+++ kdeaddons/konq-plugins/validators/validatorsplugin.desktop	Sun Apr 25 00:52:01 2004
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Website validators
+Name[ar]=أداة التأكد من صلاحية المواقع
 Name[bs]=Validatori za Web stranice
 Name[ca]=Validadors de pàgines web
 Name[cs]=Validátory webových stránek
@@ -41,6 +42,7 @@
 Name[zh_CN]=网站校验器
 Comment=CSS and HTML validator plugins
 Comment[af]=Css en Html geldighiedstoetser inprop modules
+Comment[ar]=ملحقات للتأكد من صلاحية ملفات HTML و CSS
 Comment[bg]=Приставка за валидиране на CSS и HTML
 Comment[bs]=CSS i HTML validator dodaci
 Comment[ca]=Connectors de validació CSS i HTML
@@ -66,7 +68,7 @@
 Comment[nn]=Programtillegg for validering av CSS og HTML
 Comment[pl]=Wtyczka sprawdzania poprawności HTML i CSS
 Comment[pt]='Plugins' de validação de CSS e HTML 
-Comment[pt_BR]=Plug-Ins validador para CSS e HTML
+Comment[pt_BR]=Plug-ins validadores para CSS e HTML
 Comment[ro]=Module de validare CSS şi HTML
 Comment[ru]=Модули проверки правильности HTML и CSS
 Comment[sk]=Modul pre kontrolu CSS a HTML
diff -urN -x CVS kdeaddons.orig/konq-plugins/webarchiver/archivedialog.cpp kdeaddons/konq-plugins/webarchiver/archivedialog.cpp
--- kdeaddons.orig/konq-plugins/webarchiver/archivedialog.cpp	Tue Mar 23 22:08:52 2004
+++ kdeaddons/konq-plugins/webarchiver/archivedialog.cpp	Mon Apr  5 12:56:20 2004
@@ -44,6 +44,8 @@
 
 #undef DEBUG_WAR
 
+#define CONTENT_TYPE "<meta http-equiv=\"content-type\" content=\"text/html; charset=utf-8\">"
+
 ArchiveDialog::ArchiveDialog(QWidget *parent, const QString &filename,
         KHTMLPart *part) :
         KDialogBase(parent, "WebArchiveDialog", false, "Web Archiver",
@@ -126,7 +128,7 @@
 {
    KTempFile tmpFile;
    QTextStream* textStream = tmpFile.textStream();
-   textStream->setEncoding(QTextStream::Locale);
+   textStream->setEncoding(QTextStream::UnicodeUTF8);
 
    m_widget->progressBar->setProgress(m_widget->progressBar->totalSteps());
 
@@ -175,6 +177,43 @@
    }
 }
 
+static bool hasAttribute(const DOM::Node &pNode, const QString &attrName, const QString &attrValue)
+{
+   const DOM::Element element = (const DOM::Element) pNode;
+   DOM::Attr attr;
+   DOM::NamedNodeMap attrs = element.attributes();
+   unsigned long lmap = attrs.length();
+   for( unsigned int j=0; j<lmap; j++ ) {
+      attr = static_cast<DOM::Attr>(attrs.item(j));
+      if ((attr.name().string().upper() == attrName) &&
+          (attr.value().string().upper() == attrValue))
+         return true;
+   }
+   return false;
+}
+
+static bool hasChildNode(const DOM::Node &pNode, const QString &nodeName)
+{
+   DOM::Node child;
+   try
+   {
+     // We might throw a DOM exception
+     child = pNode.firstChild();
+   }
+   catch (...)
+   {
+     // No children, stop recursion here
+     child = DOM::Node();
+   }
+
+   while(!child.isNull()) {
+     if (child.nodeName().string().upper() == nodeName)
+        return true;
+     child = child.nextSibling();
+   }
+   return false;
+}
+
 /* Transform DOM-Tree to HTML */
 
 void ArchiveDialog::saveArchiveRecursive(const DOM::Node &pNode, const KURL& baseURL,
@@ -198,6 +237,9 @@
         /* Skip BASE, everything is relative to index.html
          * Saving SCRIPT but they can cause trouble!
          */
+      } else if ((nodeName == "META") && hasAttribute(pNode, "HTTP-EQUIV", "CONTENT-TYPE")) {
+        /* Skip content-type meta tag, we provide our own.
+         */
       } else {
         if (!m_bPreserveWS) {
           if (nodeName == "PRE") {
@@ -252,8 +294,18 @@
         if (!(attributes.isEmpty())){
   	   text += " ";
         }
-     text += attributes.simplifyWhiteSpace();
-     text += ">";
+        text += attributes.simplifyWhiteSpace();
+        text += ">";
+
+        if (nodeName == "HTML") {
+          /* Search for a HEAD tag, if not found, generate one.
+           */
+          if (!hasChildNode(pNode, "HEAD"))
+            text += "\n" + strIndent + "  <HEAD>" CONTENT_TYPE "</HEAD>";
+        }
+        else if (nodeName == "HEAD") {
+          text += "\n" + strIndent + "  " + CONTENT_TYPE;
+        }
      }
    } else {
      const QString& nodeValue(pNode.nodeValue().string());
diff -urN -x CVS kdeaddons.orig/konq-plugins/webarchiver/webarchiverplugin.desktop kdeaddons/konq-plugins/webarchiver/webarchiverplugin.desktop
--- kdeaddons.orig/konq-plugins/webarchiver/webarchiverplugin.desktop	Sun Feb 29 15:42:01 2004
+++ kdeaddons/konq-plugins/webarchiver/webarchiverplugin.desktop	Sun Apr 25 00:52:03 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Name=Web archiver
 Name[af]=Web argifeerder
+Name[ar]=مؤرشف الويب
 Name[bs]=Web arhiver
 Name[ca]=Arxivador web
 Name[cs]=Webový archivátor
@@ -28,7 +29,7 @@
 Name[nso]=Palo ya Polokelo ya Web
 Name[pl]=Archiwizacja stron WWW
 Name[pt]=Arquivar web
-Name[pt_BR]=Arquivos Web
+Name[pt_BR]=Arquivador Web
 Name[ro]=Arhivator web
 Name[ru]=Веб-архиватор
 Name[sk]=Web archívy
@@ -43,6 +44,7 @@
 Name[xx]=xxWeb archiverxx
 Name[zh_CN]=万维网归档
 Comment=Creates archives of websites
+Comment[ar]=يقوم بإنشاء أرشيفات للوماقع
 Comment[bg]=Създаване на архиви от уеб сайтове
 Comment[bs]=Pravi arhive web stranica
 Comment[ca]=Crea arxius de les pàgines web
diff -urN -x CVS kdeaddons.orig/konq-plugins/webarchiver/webarchivethumbnail.desktop kdeaddons/konq-plugins/webarchiver/webarchivethumbnail.desktop
--- kdeaddons.orig/konq-plugins/webarchiver/webarchivethumbnail.desktop	Sun Mar  7 14:18:01 2004
+++ kdeaddons/konq-plugins/webarchiver/webarchivethumbnail.desktop	Sun Apr 25 00:52:05 2004
@@ -3,6 +3,7 @@
 Type=Service
 Name=Web Archives
 Name[af]=Web Argiewe
+Name[ar]=أرشيفات الويب
 Name[az]=Veb Arxivləri
 Name[bs]=Web arhive
 Name[ca]=Arxius web
diff -urN -x CVS kdeaddons.orig/konq-plugins/webarchiver/x-webarchive.desktop kdeaddons/konq-plugins/webarchiver/x-webarchive.desktop
--- kdeaddons.orig/konq-plugins/webarchiver/x-webarchive.desktop	Sun Mar  7 14:18:03 2004
+++ kdeaddons/konq-plugins/webarchiver/x-webarchive.desktop	Sun Apr 25 00:52:07 2004
@@ -2,6 +2,7 @@
 Encoding=UTF-8
 Comment=Web Archive
 Comment[af]=Web Argief
+Comment[ar]=أرشيف الويب
 Comment[az]=Veb Arxivləri
 Comment[bg]=Уеб архиви
 Comment[bs]=Web arhive
diff -urN -x CVS kdeaddons.orig/kontact/plugins/newsticker/kcmkontactknt.desktop kdeaddons/kontact/plugins/newsticker/kcmkontactknt.desktop
--- kdeaddons.orig/kontact/plugins/newsticker/kcmkontactknt.desktop	Fri Mar 12 13:41:21 2004
+++ kdeaddons/kontact/plugins/newsticker/kcmkontactknt.desktop	Sun Apr 25 00:52:09 2004
@@ -29,6 +29,7 @@
 Name[is]=Heimildir frétta
 Name[it]=Fonti delle novità
 Name[ja]=ニュース ティッカー
+Name[nb]=Nyhetskilder
 Name[nl]=Nieuwsbronnen
 Name[nn]=Nyhendekjelder
 Name[pl]=Źródła wiadomości
@@ -65,6 +66,7 @@
 Comment[is]=Stilla heimildir frétta
 Comment[it]=Impostazione fonti delle novità
 Comment[ja]=ニュース ティッカーの設定
+Comment[nb]=Oppsett av nyhetskilder
 Comment[nl]=Instellingen voor nieuwsbronnen
 Comment[nn]=Oppsett av nyhendekjelder
 Comment[pl]=Ustawienie źródeł wiadomości
@@ -95,16 +97,17 @@
 Keywords[fi]=uutisnäyttäjä, muokata, asetukset
 Keywords[fr]=news ticker, configurer, réglages
 Keywords[he]=חדשות, הגדרות, תצורה
-Keywords[hi]=न्यूज टिकर, कॉन्फिगर, विन्यास 
+Keywords[hi]=न्यूज टिकर, कॉन्फ़िगर, विन्यास 
 Keywords[hu]=hírmegjelenítő,beállítások,konfigurálás
 Keywords[is]=news ticker, configure, settings, fréttir
 Keywords[it]=news ticker,configura, impostazioni
 Keywords[ja]=ニュース ティッカー,設定,設定
+Keywords[nb]=nyhetstelegraf, oppsett, innstillinger
 Keywords[nl]=nieuws,ticker,newsticker,applet,bronbestand
 Keywords[nn]=nyhende,oppsett,innstillingar
 Keywords[pl]=wiadomości,ticker,newsticker,konfiguracja,ustawienia,źródła
 Keywords[pt]=notícias,configurar,preferências
-Keywords[pt_BR]=mostrador de notícias,configurar, preferências
+Keywords[pt_BR]=mostrador de notícias,configurar, preferências,newsticker
 Keywords[ro]=ştiri,bară,configurare,setări
 Keywords[ru]=сборка новостей, настройка, новости
 Keywords[sk]=sledovanie správ,news ticker,nastaviť
diff -urN -x CVS kdeaddons.orig/kontact/plugins/newsticker/newstickerplugin.desktop kdeaddons/kontact/plugins/newsticker/newstickerplugin.desktop
--- kdeaddons.orig/kontact/plugins/newsticker/newstickerplugin.desktop	Fri Mar 12 13:41:23 2004
+++ kdeaddons/kontact/plugins/newsticker/newstickerplugin.desktop	Sun Apr 25 00:52:11 2004
@@ -13,6 +13,7 @@
 X-KDE-PluginInfo-EnabledByDefault=true
 
 Comment=Kontact NewsTicker Plugin
+Comment[ar]=ملحق Kontact NewsTicker
 Comment[bg]=Приставка за NewsTicker
 Comment[ca]=Connector de Kontact per al teletip de notícies
 Comment[cs]=Modul novinek aplikace Kontact
@@ -30,11 +31,12 @@
 Comment[it]=Plugin per la gestione delle news per Kontact
 Comment[ja]=Kontact ニュース ティッカープラグイン
 Comment[ms]=Plug masuk NewsTicker Kontact
+Comment[nb]=Nyhetsmodul for Kontact
 Comment[nl]=Kontact NewsTicker-plugin
 Comment[nn]=Programtillegg for nyhendetelegraf i Kontact
 Comment[pl]=Wtyczka NewsTickera dla Kontact
 Comment[pt]='Plugin' de Notícias do Kontact
-Comment[pt_BR]=Plugin do Mostrador de Notícias para o Kontact
+Comment[pt_BR]=Plug-in do Mostrador de Notícias para o Kontact
 Comment[ro]=Modul bară cu ştiri pentru Kontact
 Comment[ru]=Модуль NewsTicker из пакета Kontact
 Comment[sk]=Modul pre sledovanie správ pre Kontact
@@ -56,9 +58,9 @@
 Name[hu]=Hírmegjelenítő
 Name[is]=Fréttastrimill
 Name[ja]=ニュース ティッカー
+Name[nb]=Nyhetstelegraf
 Name[nn]=Nyhendetelegraf
 Name[pt]=Notícias
-Name[pt_BR]=Mostrador de Notícias
 Name[ro]=Bară cu ştiri
 Name[ru]=Сборщик новостей NewsTicker
 Name[sk]=Sledovanie správ
diff -urN -x CVS kdeaddons.orig/ksig/ksig.desktop kdeaddons/ksig/ksig.desktop
--- kdeaddons.orig/ksig/ksig.desktop	Fri Mar 12 13:41:25 2004
+++ kdeaddons/ksig/ksig.desktop	Sun Apr 25 00:52:13 2004
@@ -29,7 +29,8 @@
 GenericName[is]=Ritill fyrir undirritanir
 GenericName[it]=Editor della firma
 GenericName[ja]=署名エディタ
-GenericName[nl]=Handtekeneningen bewerken
+GenericName[nb]=Signaturredigering
+GenericName[nl]=Handtekeningen bewerken
 GenericName[nn]=Signaturredigering
 GenericName[pl]=Edytor podpisów
 GenericName[pt]=Editor de Assinaturas
diff -urN -x CVS kdeaddons.orig/noatun-plugins/alarm/wakeup.plugin kdeaddons/noatun-plugins/alarm/wakeup.plugin
--- kdeaddons.orig/noatun-plugins/alarm/wakeup.plugin	Fri Mar 12 13:41:27 2004
+++ kdeaddons/noatun-plugins/alarm/wakeup.plugin	Sun Apr 25 00:52:15 2004
@@ -6,6 +6,7 @@
 License=GPL
 Name=WakeUp
 Name[af]=Word wakker
+Name[ar]=استيقظ
 Name[cs]=Budíček
 Name[cy]=Deffro
 Name[da]=VågnOp
@@ -34,6 +35,7 @@
 Name[zh_CN]=醒来
 Comment=A simple, customizable alarm
 Comment[af]='n eenvoudige, pakmaakbaar alarm
+Comment[ar]=منبه بسيط قابل للتخصيص
 Comment[bg]=Проста и лесна за конфигуриране аларма за събуждане сутрин
 Comment[bs]=Jednostavan, lagan za podešavanje alarm
 Comment[ca]=Una alarma senzilla i personalitzable
diff -urN -x CVS kdeaddons.orig/noatun-plugins/alsaplayer/alsaplayerui.plugin kdeaddons/noatun-plugins/alsaplayer/alsaplayerui.plugin
--- kdeaddons.orig/noatun-plugins/alsaplayer/alsaplayerui.plugin	Sun Feb 29 15:42:15 2004
+++ kdeaddons/noatun-plugins/alsaplayer/alsaplayerui.plugin	Sun Apr 25 00:52:17 2004
@@ -6,6 +6,7 @@
 License=Artistic
 Name=AlsaPlayer
 Name[af]=Alsa-speler
+Name[ar]=مشغل Alsa
 Name[cy]=ChwaraewrAlsa
 Name[hi]=अल्सा-प्लेयर
 Name[nb]=Alsaspiller
diff -urN -x CVS kdeaddons.orig/noatun-plugins/charlatan/charlatanui.plugin kdeaddons/noatun-plugins/charlatan/charlatanui.plugin
--- kdeaddons.orig/noatun-plugins/charlatan/charlatanui.plugin	Sun Feb 29 15:42:19 2004
+++ kdeaddons/noatun-plugins/charlatan/charlatanui.plugin	Sun Apr 25 00:52:19 2004
@@ -10,7 +10,6 @@
 Name[hi]=चार्लटन
 Name[nn]=Sjarlatan
 Name[pl]=Szarlatan
-Name[pt_BR]=Charlatão
 Name[ro]=Şarlatan
 Name[ru]=Обманщик
 Name[ta]=சர்லதான்
@@ -40,6 +39,7 @@
 Comment[pl]=Interfejs martwego człowieka
 Comment[pt]=A interface do homem morto
 Comment[pt_BR]=Uma interface para homens sem saída
+Comment[ro]=Interfaţa "dead man"
 Comment[ru]=Интерфейс Обманщик
 Comment[sk]=Rozhranie dead man
 Comment[sl]=Vmesnik mrtvega človeka
diff -urN -x CVS kdeaddons.orig/noatun-plugins/dub/dub/dub.plugin kdeaddons/noatun-plugins/dub/dub/dub.plugin
--- kdeaddons.orig/noatun-plugins/dub/dub/dub.plugin	Mon Mar  8 13:46:46 2004
+++ kdeaddons/noatun-plugins/dub/dub/dub.plugin	Sun Apr 18 14:32:19 2004
@@ -36,7 +36,7 @@
 Comment[nn]=Dub-programtillegg for speleliste
 Comment[pl]=Wtyczka listy nagrań Dub
 Comment[pt]='Plugin' da Lista Dub
-Comment[pt_BR]=Plugin de Lista de reprodução do Dub
+Comment[pt_BR]=Plug-in de Lista de Músicas do Dub
 Comment[ro]=Mod de listă de redare Dub
 Comment[ru]=Модуль списка произведений Dub
 Comment[sk]=Modul pre Dub playlist
diff -urN -x CVS kdeaddons.orig/noatun-plugins/ffrs/ffrs.plugin kdeaddons/noatun-plugins/ffrs/ffrs.plugin
--- kdeaddons.orig/noatun-plugins/ffrs/ffrs.plugin	Mon Mar  8 13:46:48 2004
+++ kdeaddons/noatun-plugins/ffrs/ffrs.plugin	Sun Apr 25 00:52:21 2004
@@ -60,6 +60,7 @@
 Comment[pl]=Pływające soczewki
 Comment[pt]=Osciloscópios Flutuantes
 Comment[pt_BR]=Escopos Flutuantes
+Comment[ro]=Vizualizări grafice care plutesc
 Comment[ru]=Зрительный образ Плавающие кривые
 Comment[sk]=Plávajúce rámce
 Comment[sl]=Plavajoča območja
diff -urN -x CVS kdeaddons.orig/noatun-plugins/jefferson/jefferson.plugin kdeaddons/noatun-plugins/jefferson/jefferson.plugin
--- kdeaddons.orig/noatun-plugins/jefferson/jefferson.plugin	Sun Feb 29 15:42:25 2004
+++ kdeaddons/noatun-plugins/jefferson/jefferson.plugin	Sun Apr 25 00:52:23 2004
@@ -13,6 +13,7 @@
 Name[xx]=xxJeffersonxx
 Comment=Displays Noatun status directly on screen
 Comment[af]=Vertoon Noatun status direk op skerm
+Comment[ar]=يظهر حالة Noatun مباشرة على الشاشة
 Comment[bg]=Показване на състоянието на Noatun директно на екрана
 Comment[ca]=Mostra l'estat del Noatun directament a la pantalla
 Comment[cs]=Zobrazuje stav Noatunu přímo na obrazovce
diff -urN -x CVS kdeaddons.orig/noatun-plugins/luckytag/luckytag.plugin kdeaddons/noatun-plugins/luckytag/luckytag.plugin
--- kdeaddons.orig/noatun-plugins/luckytag/luckytag.plugin	Mon Mar  8 13:46:50 2004
+++ kdeaddons/noatun-plugins/luckytag/luckytag.plugin	Sun Apr 25 00:52:25 2004
@@ -5,6 +5,7 @@
 License=BSD
 Name=Lucky Tag Reader
 Name[af]=Gelukkig Etiket Leser
+Name[ar]=قارئ العلامات المحظوظ
 Name[ca]=Lector d'etiquetes Lucky
 Name[cs]=Chytré čtení tagů
 Name[cy]=Darllenydd Tag Lwcus
@@ -42,6 +43,7 @@
 Name[zh_CN]=幸运标签读取器
 Comment=Guesses tag information from the filename
 Comment[af]=Raaiskote etiket informasie van die lêernaam
+Comment[ar]=يحزر معلومات العلامات من اسم الملف
 Comment[bg]=Отгатване на информацията за песните от името на файла
 Comment[ca]=Intenta endevinar el contingut de l'etiqueta a partir del nom del fitxer
 Comment[cs]=Odhaduje informace o značkách z názvu souboru
@@ -54,7 +56,7 @@
 Comment[fi]=Arvaa tagitiedot suoraan tiedostonimestä
 Comment[fr]=Devine les informations de balises à partir du nom de fichier
 Comment[he]=ניחוש מידע התגיות מתוך שם הקובץ
-Comment[hi]=फाइल-नामों से टैग जानकारी का अंदाजा लगाता है
+Comment[hi]=फ़ाइल-नामों से टैग जानकारी का अंदाजा लगाता है
 Comment[hu]=Tag-információ a fájlnév alapján
 Comment[is]=Giskar á lagamerkingar útfrá skráarheitum
 Comment[it]=Indovina le informazioni dei tag dal nome del file
diff -urN -x CVS kdeaddons.orig/noatun-plugins/lyrics/lyrics.plugin kdeaddons/noatun-plugins/lyrics/lyrics.plugin
--- kdeaddons.orig/noatun-plugins/lyrics/lyrics.plugin	Sun Feb 29 15:42:29 2004
+++ kdeaddons/noatun-plugins/lyrics/lyrics.plugin	Sun Apr 25 00:52:27 2004
@@ -6,6 +6,7 @@
 License=BSD
 Name=Lyrics
 Name[af]=Lirieke
+Name[ar]=الكلمات
 Name[cs]=Text skladby
 Name[cy]=Geiriau cân
 Name[da]=Sangtekst
@@ -36,6 +37,7 @@
 Name[zh_CN]=歌词
 Comment=Display the lyrics for the current song
 Comment[af]=Vertoon die lirieke vir die huidige liedjie
+Comment[ar]=يعرض كلمات الأغنية الحالية
 Comment[bg]=Показване на текста на текущата песента (лириката)
 Comment[ca]=Mostra la lletra de la cançó actual
 Comment[cs]=Zobrazuje slova k současné skladbě
diff -urN -x CVS kdeaddons.orig/noatun-plugins/noatunmadness/madness.plugin kdeaddons/noatun-plugins/noatunmadness/madness.plugin
--- kdeaddons.orig/noatun-plugins/noatunmadness/madness.plugin	Sun Feb 29 15:42:33 2004
+++ kdeaddons/noatun-plugins/noatunmadness/madness.plugin	Sun Apr 25 00:52:29 2004
@@ -5,6 +5,7 @@
 Type=visualization
 License=GPL
 Name=Madness
+Name[ar]=جنون
 Name[cs]=Šílenost
 Name[cy]=Gwallgofrwydd
 Name[da]=Galskab
@@ -19,7 +20,6 @@
 Name[nso]=Bogafa
 Name[pl]=Szaleństwo
 Name[pt]=Loucura
-Name[pt_BR]=Maldade
 Name[ro]=Nebunie
 Name[ru]=Безумие
 Name[sl]=Norost
@@ -32,6 +32,7 @@
 Name[zh_CN]=疯狂
 Comment=A weird and annoying scope that moves your windows
 Comment[af]='n weird en irriterend skoop wat skuiwe jou vensters
+Comment[ar]=ملحق غريب و مزعج يقوم بتحريك نوافذك
 Comment[bg]=Досадна приставка за преместване на прозореца
 Comment[ca]=Un estrany i molest calidoscopi que mou les finestres
 Comment[cs]=Otravná vizualizace, který hýbe s vašimi okny
@@ -55,6 +56,7 @@
 Comment[pl]=Dziwna i irytująca soczewka, która przesuwa twoje okna
 Comment[pt]=Um osciloscópio estranho e irritante que move as suas janelas
 Comment[pt_BR]= Um escopo estranho que move suas janelas
+Comment[ro]=Un modul de vizualizare care mişcă ferestrele
 Comment[ru]=Странный зрительный образ, перемещающий ваши окна
 Comment[sk]=Šialený a protivný rámec ktorý hýbe vašími oknami.
 Comment[sl]=Čudno in nadležno območje, ki premika vaša okna
diff -urN -x CVS kdeaddons.orig/noatun-plugins/oblique/oblique.plugin kdeaddons/noatun-plugins/oblique/oblique.plugin
--- kdeaddons.orig/noatun-plugins/oblique/oblique.plugin	Sun Feb 29 15:42:35 2004
+++ kdeaddons/noatun-plugins/oblique/oblique.plugin	Sun Apr 25 00:52:31 2004
@@ -7,7 +7,6 @@
 Name=Oblique
 Name[cy]=Lletraws
 Name[hi]=ऑब्लिक
-Name[pt_BR]=Oblíquo
 Name[ro]=Oblic
 Name[ru]=Наклони
 Name[sl]=Ležeče
@@ -15,6 +14,7 @@
 Name[tr]=Eğik
 Name[xx]=xxObliquexx
 Comment=Auto-Collating Playlist
+Comment[ar]=ترتيب تلقائي لقائمة التشغيل
 Comment[bg]=Списък на файлове за изпълнение, който автоматично се сортира и проверява
 Comment[ca]=Ordenació automàtica de les llistes de reproducció
 Comment[cs]=Samořadící seznam skladeb
@@ -30,10 +30,12 @@
 Comment[is]=Sjálfvirkur lagalisti
 Comment[it]=Raccogli automaticamente la lista di selezione
 Comment[ja]=プレイリストの自動照合
+Comment[nb]=Selvjusterende spilleliste
 Comment[nl]=Auto-plakkende afspeellijst
 Comment[pl]=Automatycznie ustawiająca się lista odtwarzania
 Comment[pt]=Lista de Músicas Autocolante
-Comment[pt_BR]=Lista de reprodução de Colagem Automática
+Comment[pt_BR]=Lista de Músicas de Colagem Automática
+Comment[ro]=Lista cu ordonare automată
 Comment[ru]=Объедиение списков воспроизведения
 Comment[sk]=Automaticky triedený playlist
 Comment[sl]=Samodejno zlagajoč predvajalni seznam
diff -urN -x CVS kdeaddons.orig/noatun-plugins/pitchablespeed/pitchablespeed.plugin kdeaddons/noatun-plugins/pitchablespeed/pitchablespeed.plugin
--- kdeaddons.orig/noatun-plugins/pitchablespeed/pitchablespeed.plugin	Mon Mar  8 13:46:52 2004
+++ kdeaddons/noatun-plugins/pitchablespeed/pitchablespeed.plugin	Mon Apr 12 18:49:19 2004
@@ -53,12 +53,12 @@
 Comment[he]=מאפשר התאמה של מהירות הניגון בעזרת מחוון
 Comment[hi]=प्लेइंग स्पीड एक स्लाइडर के द्वारा समायोजन योग्य बनाता है
 Comment[hu]=Változtatható lejátszási sebességet tesz lehetővé
-Comment[is]=Gerir afspilunarhraðann stillanlegann með rennislá
+Comment[is]=Gerir afspilunarhraðan stillanlegan með rennislá
 Comment[it]=Permette di modificare la velocità di riproduzione tramite una barra
 Comment[ja]=再生スピードをスライダーで調節させる
 Comment[lt]=Leidžia keisti grojimo greitį su šliaužikliu
 Comment[ms]=Menjadikan kelajuan pemain boleh laras dengan slider
-Comment[nb]=Gjør avspillilngsfarten justerbar med en skyver
+Comment[nb]=Gjør avspillingsfarten justerbar med en skyver
 Comment[nl]=Maakt de afspeelsnelheid instelbaar met een schuifregelaar
 Comment[pl]=Umożliwia łatwą zmianę szybkości odtwarzania
 Comment[pt]=Torna a velocidade de reprodução ajustável com uma barra
diff -urN -x CVS kdeaddons.orig/noatun-plugins/synaescope/synaescope.plugin kdeaddons/noatun-plugins/synaescope/synaescope.plugin
--- kdeaddons.orig/noatun-plugins/synaescope/synaescope.plugin	Sun Feb 29 15:42:39 2004
+++ kdeaddons/noatun-plugins/synaescope/synaescope.plugin	Sun Apr 25 00:52:33 2004
@@ -14,6 +14,7 @@
 Name[xx]=xxSynaescopexx
 Comment=Noatun visualization
 Comment[af]=Noatun visualiseering
+Comment[ar]=مرئيات Noatun
 Comment[bg]=Приставка за визуализация за Noatun
 Comment[ca]=Visualització en el Noatun
 Comment[cs]=Vizualizace pro Noatun
diff -urN -x CVS kdeaddons.orig/noatun-plugins/tippercanoe/tippecanoe.plugin kdeaddons/noatun-plugins/tippercanoe/tippecanoe.plugin
--- kdeaddons.orig/noatun-plugins/tippercanoe/tippecanoe.plugin	Sun Feb 29 15:42:41 2004
+++ kdeaddons/noatun-plugins/tippercanoe/tippecanoe.plugin	Sun Apr 25 00:52:35 2004
@@ -12,6 +12,7 @@
 Name[xx]=xxTippecanoexx
 Comment=A visualization you can play with for hours.
 Comment[af]='n visualiseering jy kan speel met vir ure.
+Comment[ar]=مرئية يمكنك اللعب بها لساعات.
 Comment[bg]=Забавна приставка за визуализация за Noatun
 Comment[ca]=Un visualitzador que pot reproduir durant hores.
 Comment[cs]=Vizualizace, se kterou si vyhrajete
@@ -35,7 +36,7 @@
 Comment[nl]=Een visualisatie waarmee u uren kunt spelen.
 Comment[pl]=Wizualizacja, którą można się bawić godzinami.
 Comment[pt]=Uma visualização que o utilizador pode reproduzir durante horas.
-Comment[pt_BR]=Uma visualização que você pode reproduzir. 
+Comment[pt_BR]=Uma visualização com a qual você pode reproduzir por horas. 
 Comment[ro]=Un modul de vizualizare cu care vă puteţi juca ore întregi
 Comment[ru]=Зрительный образ, который вы можете рассматривать часами
 Comment[sk]=Vizualizácia s ktorou sa môžete hrať celé hodiny.
diff -urN -x CVS kdeaddons.orig/noatun-plugins/tyler/tyler.plugin kdeaddons/noatun-plugins/tyler/tyler.plugin
--- kdeaddons.orig/noatun-plugins/tyler/tyler.plugin	Sun Feb 29 15:42:43 2004
+++ kdeaddons/noatun-plugins/tyler/tyler.plugin	Sun Apr 25 00:52:37 2004
@@ -12,6 +12,7 @@
 Name[ta]=டைலர்
 Name[xx]=xxTylerxx
 Comment=Ported from Infinity from XMMS
+Comment[ar]=منقولة من XMMS Infinity
 Comment[bg]=Приставка, която наподобява темата Infinity от плеъра XMMS
 Comment[ca]=Portat des de l'Infinity de l'XMMS
 Comment[cs]=Přenesený modul Infinity z XMMS
diff -urN -x CVS kdeaddons.orig/noatun-plugins/wavecapture/wavecapture.plugin kdeaddons/noatun-plugins/wavecapture/wavecapture.plugin
--- kdeaddons.orig/noatun-plugins/wavecapture/wavecapture.plugin	Sun Feb 29 15:42:45 2004
+++ kdeaddons/noatun-plugins/wavecapture/wavecapture.plugin	Sun Apr 25 00:52:39 2004
@@ -51,7 +51,7 @@
 Comment[fr]=Capturer vers un fichier wave
 Comment[gl]=Captura sonora nun ficheiro
 Comment[he]=לכידה לקובץ wave
-Comment[hi]=वेव फाइल में केप्चर करता है
+Comment[hi]=वेव फ़ाइल में केप्चर करता है
 Comment[hu]=Mentés WAV fájlba
 Comment[is]=Vista hljóð í wave skrár
 Comment[it]=Cattura su di file wave
@@ -61,7 +61,7 @@
 Comment[nl]=Opnemen als wave-bestand
 Comment[pl]=Zgrywanie do pliku WAV
 Comment[pt]=Capturar para um ficheiro WAVE
-Comment[pt_BR]=captura e transforma para arquivos wave
+Comment[pt_BR]=Captura e transforma para arquivos wave
 Comment[ro]=Capturează sunetul într-un fişier WAV
 Comment[ru]=Сохранение произведения в формате wav
 Comment[sk]=Uložiť wave do súboru
diff -urN -x CVS kdeaddons.orig/renamedlgplugins/audio/renaudiodlg.desktop kdeaddons/renamedlgplugins/audio/renaudiodlg.desktop
--- kdeaddons.orig/renamedlgplugins/audio/renaudiodlg.desktop	Fri Mar 12 13:41:31 2004
+++ kdeaddons/renamedlgplugins/audio/renaudiodlg.desktop	Sun Apr 25 00:52:41 2004
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Type=Service
 Name=Audio Preview
+Name[ar]=معاينة صوتية
 Name[ca]=Escolta una mostra de so
 Name[cs]=Náhled zvuku
 Name[cy]=Rhagolwg Sain
@@ -19,6 +20,7 @@
 Name[is]=Hljóðforsýn
 Name[it]=Anteprima audio
 Name[ja]=オーディオプレビュー
+Name[nb]=Lydforhåndsvisning
 Name[nl]=Audiovoorbeeld
 Name[pl]=Podgląd plików audio
 Name[pt]=Previsão Áudio
diff -urN -x CVS kdeaddons.orig/renamedlgplugins/images/renimagedlg.desktop kdeaddons/renamedlgplugins/images/renimagedlg.desktop
--- kdeaddons.orig/renamedlgplugins/images/renimagedlg.desktop	Sun Feb 29 15:42:49 2004
+++ kdeaddons/renamedlgplugins/images/renimagedlg.desktop	Sun Apr 25 00:52:43 2004
@@ -1,6 +1,7 @@
 [Desktop Entry]
 Type=Service
 Name=Image Displayer
+Name[ar]=عارض الصور
 Name[bs]=Preglednik slika
 Name[ca]=Visor d'imatges
 Name[cs]=Prohlížeč obrázků
@@ -16,7 +17,7 @@
 Name[fr]=Afficheur d'images
 Name[gl]=Amosador de Imaxes
 Name[he]=מציג תמונות
-Name[hi]=चित्र प्रदर्शक
+Name[hi]=छवि प्रदर्शक
 Name[hr]=Preglednik slika
 Name[hu]=Képmegjelenítő
 Name[is]=Myndsjá
diff -urN -x CVS kdeaddons.orig/vimpart/kcmvim/kcmvim.desktop kdeaddons/vimpart/kcmvim/kcmvim.desktop
--- kdeaddons.orig/vimpart/kcmvim/kcmvim.desktop	Fri Mar 12 13:41:33 2004
+++ kdeaddons/vimpart/kcmvim/kcmvim.desktop	Sun Apr 25 00:52:45 2004
@@ -8,6 +8,7 @@
 X-KDE-FactoryName=vim
 
 Name=Vim Component Configuration
+Name[ar]=تهيئة مكون Vim
 Name[ca]=Configuració del component de Vim
 Name[cs]=Nastavení Vim komponenty
 Name[cy]=Ffurfweddiad Cydrannau Vim
@@ -20,11 +21,12 @@
 Name[fr]=Configuration du composant Vim
 Name[ga]=Cumraíocht Chomhpháirt Vim
 Name[he]= הגדרות רכיב Vim
-Name[hi]=वीआईएम अवयव कॉन्फिगरेशन
+Name[hi]=वीआईएम अवयव कॉन्फ़िगरेशन
 Name[hu]=Vim-komponens
 Name[is]=Stillingar íhluts fyrir vim
 Name[it]=Configurazione componente Vim
 Name[ja]=Vimコンポーネントの設定
+Name[nb]=Vim komponentoppsett
 Name[nl]=Configuratie van VIM-component
 Name[pl]=Komponent Vim
 Name[pt]=Configuração do Componente Vim
diff -urN -x CVS kdeaddons.orig/vimpart/src/vimpart.desktop kdeaddons/vimpart/src/vimpart.desktop
--- kdeaddons.orig/vimpart/src/vimpart.desktop	Fri Mar 12 13:41:35 2004
+++ kdeaddons/vimpart/src/vimpart.desktop	Sun Apr 25 00:52:47 2004
@@ -1,5 +1,6 @@
 [Desktop Entry]
 Name=Embedded Vim Component
+Name[ar]=مكون Vim المدمج
 Name[ca]=Component encastat de Vim
 Name[cs]=Pohltitelná Vim komponenta
 Name[cy]=Cydran Vim Mewnadeiledig
@@ -17,6 +18,7 @@
 Name[is]=Íhlutur í vim
 Name[it]=Componente integrato Vim
 Name[ja]=埋め込み可能なVimコンポーネント
+Name[nb]=Innebygget Vim-komponent
 Name[nl]=Ingebed VIM-component
 Name[pl]=Osadzony komponent Vim
 Name[pt]=Componente Vim Embebido
